<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2021 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        child
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
sample:
  genomeAssembly: &quot;hg38&quot;
  vcf: &quot;/home/ivanwilliamharsono/Downloads/trio_patient/annotated/V350145665_L04_B5EHOMdmhwXAAAA-515-SnpEff-dbSNP-ClinVar-dbNSFP_annotated-deepTrio.vcf.gz&quot;
  proband: &quot;child&quot;
  hpoIds:
  - &quot;HP:0000007&quot;
  - &quot;HP:0001433&quot;
  - &quot;HP:0001903&quot;
  - &quot;HP:0001541&quot;
  - &quot;HP:0010972&quot;
  - &quot;HP:0005561&quot;
  - &quot;HP:0001396&quot;
  - &quot;HP:0001410&quot;
  - &quot;HP:0002910&quot;
  - &quot;HP:0001399&quot;
  - &quot;HP:0000938&quot;
  - &quot;HP:0001653&quot;
  - &quot;HP:0200114&quot;
  - &quot;HP:0003073&quot;
  - &quot;HP:0003233&quot;
  - &quot;HP:0001873&quot;
  - &quot;HP:0002151&quot;
  - &quot;HP:0031964&quot;
  - &quot;HP:0031956&quot;
  - &quot;HP:0002366&quot;
  - &quot;HP:0006568&quot;
  - &quot;HP:0004333&quot;
  - &quot;HP:0001531&quot;
  pedigree:
    persons:
    - familyId: &quot;1&quot;
      individualId: &quot;child&quot;
      paternalId: &quot;father&quot;
      maternalId: &quot;mother&quot;
      sex: &quot;MALE&quot;
      affectedStatus: &quot;AFFECTED&quot;
    - familyId: &quot;1&quot;
      individualId: &quot;father&quot;
      sex: &quot;MALE&quot;
      affectedStatus: &quot;UNAFFECTED&quot;
    - familyId: &quot;1&quot;
      individualId: &quot;mother&quot;
      sex: &quot;FEMALE&quot;
      affectedStatus: &quot;UNAFFECTED&quot;
  sex: &quot;MALE&quot;
  age: {}
analysis:
  inheritanceModes:
    AUTOSOMAL_RECESSIVE_COMP_HET: 2.0
    MITOCHONDRIAL: 0.2
    AUTOSOMAL_RECESSIVE_HOM_ALT: 0.1
    X_DOMINANT: 0.1
    AUTOSOMAL_DOMINANT: 0.1
    X_RECESSIVE_COMP_HET: 2.0
    X_RECESSIVE_HOM_ALT: 0.1
  frequencySources:
  - &quot;THOUSAND_GENOMES&quot;
  - &quot;TOPMED&quot;
  - &quot;UK10K&quot;
  - &quot;ESP_AFRICAN_AMERICAN&quot;
  - &quot;ESP_EUROPEAN_AMERICAN&quot;
  - &quot;ESP_ALL&quot;
  - &quot;EXAC_AFRICAN_INC_AFRICAN_AMERICAN&quot;
  - &quot;EXAC_AMERICAN&quot;
  - &quot;EXAC_EAST_ASIAN&quot;
  - &quot;EXAC_FINNISH&quot;
  - &quot;EXAC_NON_FINNISH_EUROPEAN&quot;
  - &quot;EXAC_OTHER&quot;
  - &quot;EXAC_SOUTH_ASIAN&quot;
  - &quot;GNOMAD_E_AFR&quot;
  - &quot;GNOMAD_E_AMR&quot;
  - &quot;GNOMAD_E_EAS&quot;
  - &quot;GNOMAD_E_FIN&quot;
  - &quot;GNOMAD_E_NFE&quot;
  - &quot;GNOMAD_E_OTH&quot;
  - &quot;GNOMAD_E_SAS&quot;
  - &quot;GNOMAD_G_AFR&quot;
  - &quot;GNOMAD_G_AMR&quot;
  - &quot;GNOMAD_G_EAS&quot;
  - &quot;GNOMAD_G_FIN&quot;
  - &quot;GNOMAD_G_NFE&quot;
  - &quot;GNOMAD_G_OTH&quot;
  - &quot;GNOMAD_G_SAS&quot;
  pathogenicitySources:
  - &quot;REVEL&quot;
  - &quot;MVP&quot;
  steps:
  - failedVariantFilter: {}
  - variantEffectFilter:
      remove:
      - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
      - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
      - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
      - &quot;FIVE_PRIME_UTR_INTRON_VARIANT&quot;
      - &quot;THREE_PRIME_UTR_INTRON_VARIANT&quot;
      - &quot;NON_CODING_TRANSCRIPT_EXON_VARIANT&quot;
      - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
      - &quot;UPSTREAM_GENE_VARIANT&quot;
      - &quot;DOWNSTREAM_GENE_VARIANT&quot;
      - &quot;INTERGENIC_VARIANT&quot;
      - &quot;REGULATORY_REGION_VARIANT&quot;
  - frequencyFilter:
      maxFrequency: 2.0
  - pathogenicityFilter:
      keepNonPathogenic: true
  - inheritanceFilter: {}
  - omimPrioritiser: {}
  - hiPhivePrioritiser:
      runParams: &quot;human, mouse, fish, ppi&quot;
  - phenixPrioritiser: {}
outputOptions:
  outputContributingVariantsOnly: true
  numGenes: 50
  outputFormats:
  - &quot;HTML&quot;
  - &quot;VCF&quot;
  - &quot;TSV_GENE&quot;
  - &quot;TSV_VARIANT&quot;
  - &quot;JSON&quot;
  outputFileName: &quot;V350145665_L04_B5EHOMdmhwXAAAA-515-exomiser-trio&quot;
  outputDirectory: &quot;/home/ivanwilliamharsono/Downloads/trio_patient/exomiser&quot;
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Failed previous VCF filters</b></td>
                    <td>
                        
                    </td>
                    <td>124</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, FIVE_PRIME_UTR_INTRON_VARIANT, THREE_PRIME_UTR_INTRON_VARIANT, NON_CODING_TRANSCRIPT_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT, REGULATORY_REGION_VARIANT]</ul>
                    </td>
                    <td>124</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>124</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>124</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Genes filtered for compatibility with AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL inheritance.</ul>
                    </td>
                    <td>66</td>
                    <td>0</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>child</th>
                    
                        <th>father</th>
                    
                        <th>mother</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>1</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>67</td>
                    
                        <td>27</td>
                    
                        <td>31</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>1</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>2</td>
                    
                        <td>0</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>12</td>
                    
                        <td>7</td>
                    
                        <td>6</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>40</td>
                    
                        <td>11</td>
                    
                        <td>23</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GBE1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000114480">GBE1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.930</b>
                    <data>(p=9.5E-4)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.658</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.985</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.640 to <a href="http://www.omim.org/entry/232500">Glycogen storage disease IV</a> associated with GBE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001903, Anemia - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001541, Ascites - HP:0001541, Ascites</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001396, Cholestasis - HP:0001399, Hepatic failure</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0001409, Portal hypertension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000969, Edema</dd><dd>HP:0003073, Hypoalbuminemia - HP:0040081, Abnormal circulating creatine kinase concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0040081, Abnormal circulating creatine kinase concentration</dd><dd>HP:0001873, Thrombocytopenia - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0002151, Increased serum lactate - HP:0040081, Abnormal circulating creatine kinase concentration</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001394, Cirrhosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Phenotypic similarity 0.658 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GBE1">GBE1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0003333, liver fibrosis</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0003333, liver fibrosis</dd><dd>HP:0001410, Decreased liver function - MP:0003333, liver fibrosis</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0003333, liver fibrosis</dd><dd>HP:0001399, Hepatic failure - MP:0003333, liver fibrosis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0010090, increased circulating creatine kinase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0010090, increased circulating creatine kinase level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0003333, liver fibrosis</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0003333, liver fibrosis</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0014074, increased brain glycogen level</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0010400, increased liver glycogen level</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GBE1%0DPGM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PGM1</a> and phenotypic similarity 0.745 to <a href="http://www.omim.org/entry/614921">Congenital disorder of glycosylation, type It</a> associated with PGM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0002240, Hepatomegaly</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0001406, Intrahepatic cholestasis</dd><dd>HP:0001410, Decreased liver function - HP:0001406, Intrahepatic cholestasis</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0001406, Intrahepatic cholestasis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0031628, Aborted sudden cardiac death</dd><dd>HP:0200114, Metabolic alkalosis - HP:0002047, Malignant hyperthermia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003236, Elevated circulating creatine kinase concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003236, Elevated circulating creatine kinase concentration</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0001943, Hypoglycemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0031964, Elevated circulating alanine aminotransferase concentration</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0031956, Elevated circulating aspartate aminotransferase concentration</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0006568, Increased hepatic glycogen content</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001680, Coarctation of aorta</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GBE1%0DPGM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PGM1</a> and phenotypic similarity 0.163 to mouse mutant of PGM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0003232, abnormal forebrain development</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 1.57 (p-value: 0.000060)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/232500">OMIM:232500</a> Glycogen storage disease IV - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/263570">OMIM:263570</a> Polyglucosan body disease, adult form - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=206583">ORPHA:206583</a> Adult polyglucosan body disease - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.930</b>
                                    <data>(p=9.5E-4)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.658</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.985</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.640 to OMIM:232500 Glycogen storage disease IV</dt>
                                
                                    <dt>Phenotypic similarity 0.476 to OMIM:263570 Polyglucosan body disease, adult form</dt>
                                
                                    <dt>Phenotypic similarity 0.203 to ORPHA:206583 Adult polyglucosan body disease</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-81648954-C-G [1/1:0/1:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting, PP3_Moderate, PP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.985</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429644.7">GBE1:ENST00000429644.7:c.593G&gt;C:p.(Arg198Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9849782</dd>
                                    
                                    <dd>REVEL: 0.873</dd>
                                    <dd>MVP: 0.985</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MATN2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000132561">MATN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.400</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.837</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MATN2%0DTWSG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TWSG1</a> and phenotypic similarity 0.633 to mouse mutant of TWSG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0008479, decreased spleen white pulp amount</dd><dd>HP:0001903, Anemia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0002398, abnormal bone marrow cell morphology/development</dd><dd>HP:0001396, Cholestasis - MP:0003644, thymus atrophy</dd><dd>HP:0001410, Decreased liver function - MP:0009541, increased thymocyte apoptosis</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0009541, increased thymocyte apoptosis</dd><dd>HP:0001399, Hepatic failure - MP:0009541, increased thymocyte apoptosis</dd><dd>HP:0000938, Osteopenia - MP:0020010, decreased bone mineral density of femur</dd><dd>HP:0001653, Mitral regurgitation - MP:0001634, internal hemorrhage</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0009541, increased thymocyte apoptosis</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0009541, increased thymocyte apoptosis</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0031109, subarachnoid hemorrhage</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000704, abnormal thymus development</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000220, increased monocyte cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MATN2%0DTWSG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TWSG1</a> and phenotypic similarity 0.179 to fish mutant of TWSG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0104386, caudal artery increased distribution, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.400</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.837</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-98003762-C-A [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs371094213">rs371094213</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.858</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254898.7">MATN2:ENST00000254898.7:c.1306C&gt;A:p.(Pro436Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9541492</dd>
                                    
                                    <dd>REVEL: 0.282</dd>
                                    <dd>MVP: 0.954</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0212%</dd>
                                    <dd>ExAC AFR: 0.0205%</dd>
                                    <dd>ExAC EAS: 0.2555%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.3653%</dd>
                                    <dd>gnomAD_E_OTH: 0.1096%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_EAS: 0.5556%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-98007571-G-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs372608950">rs372608950</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.816</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000254898.7">MATN2:ENST00000254898.7:c.1543G&gt;A:p.(Val515Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8298366</dd>
                                    
                                    <dd>REVEL: 0.122</dd>
                                    <dd>MVP: 0.830</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0042%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.0135%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC7A10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130876">SLC7A10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.395</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.832</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.265 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC7A10">SLC7A10</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC7A10%0DSLC7A7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC7A7</a> and phenotypic similarity 0.791 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=470">Lysinuric protein intolerance</a> associated with SLC7A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001903, Anemia - HP:0001903, Anemia</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001903, Anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0031020, Bone marrow hypercellularity</dd><dd>HP:0001396, Cholestasis - HP:0001399, Hepatic failure</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0000938, Osteopenia</dd><dd>HP:0001653, Mitral regurgitation - HP:0001892, Abnormal bleeding</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001987, Hyperammonemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003281, Increased circulating ferritin concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003233, Decreased HDL cholesterol concentration</dd><dd>HP:0001873, Thrombocytopenia - HP:0001873, Thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - HP:0003348, Hyperalaninemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0012280, Hepatic amyloidosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0012156, Hemophagocytosis</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC7A10%0DSLC7A7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC7A7</a> and phenotypic similarity 0.518 to mouse mutant of SLC7A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0004953, decreased spleen weight</dd><dd>HP:0001903, Anemia - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000219, increased neutrophil cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0005016, decreased lymphocyte cell number</dd><dd>HP:0001396, Cholestasis - MP:0000597, delayed hepatic development</dd><dd>HP:0001410, Decreased liver function - MP:0000597, delayed hepatic development</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000597, delayed hepatic development</dd><dd>HP:0001399, Hepatic failure - MP:0000597, delayed hepatic development</dd><dd>HP:0000938, Osteopenia - MP:0020137, decreased bone mineralization</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0000219, increased neutrophil cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000597, delayed hepatic development</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000597, delayed hepatic development</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000597, delayed hepatic development</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0008489, slow postnatal weight gain</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC7A10%0DSLC7A7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC7A7</a> and phenotypic similarity 0.323 to fish mutant of SLC7A7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0001903, Anemia - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0016849, microglia development decreased occurrence, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.395</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.832</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-33215803-C-T [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs567938130">rs567938130</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.895</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253188.8">SLC7A10:ENST00000253188.8:c.322G&gt;A:p.(Ala108Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9017956</dd>
                                    
                                    <dd>REVEL: 0.401</dd>
                                    <dd>MVP: 0.902</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0312%</dd>
                                    <dd>TOPMed: 0.0110%</dd>
                                    <dd>ExAC NFE: 0.0535%</dd>
                                    <dd>ExAC SAS: 0.0247%</dd>
                                    <dd>gnomAD_E_AMR: 0.0112%</dd>
                                    <dd>gnomAD_E_EAS: 0.0162%</dd>
                                    <dd>gnomAD_E_NFE: 0.0056%</dd>
                                    <dd>gnomAD_E_SAS: 0.0164%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>19-33210906-T-C [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs375747229">rs375747229</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.770</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000253188.8">SLC7A10:ENST00000253188.8:c.1017-8A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0625%</dd>
                                    <dd>TOPMed: 0.0068%</dd>
                                    <dd>ExAC AFR: 0.0108%</dd>
                                    <dd>ExAC EAS: 0.2005%</dd>
                                    <dd>ExAC SAS: 0.0621%</dd>
                                    <dd>gnomAD_E_EAS: 0.1803%</dd>
                                    <dd>gnomAD_E_OTH: 0.0184%</dd>
                                    <dd>gnomAD_E_SAS: 0.0455%</dd>
                                    <dd>gnomAD_G_EAS: 0.2469%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MUC12" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000205277">MUC12</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.334</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC12%0DGALNT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GALNT1</a> and phenotypic similarity 0.630 to mouse mutant of GALNT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0001903, Anemia - MP:0002607, decreased basophil cell number</dd><dd>HP:0001541, Ascites - MP:0005606, increased bleeding time</dd><dd>HP:0010972, Anemia of inadequate production - MP:0005012, decreased eosinophil cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0002607, decreased basophil cell number</dd><dd>HP:0001396, Cholestasis - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0001410, Decreased liver function - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0001399, Hepatic failure - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0006047, aortic valve regurgitation</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0005419, decreased circulating serum albumin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0005419, decreased circulating serum albumin level</dd><dd>HP:0001873, Thrombocytopenia - MP:0002607, decreased basophil cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0010470, dilated ascending aorta</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.334</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>7-100969688-A-G [0/1:./.:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs73168328">rs73168328</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536621.6">MUC12:ENST00000536621.6:c.66A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.017</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.433</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>7-100969688-A-G [0/1:./.:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs73168328">rs73168328</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536621.6">MUC12:ENST00000536621.6:c.66A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-100991252-A-C [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs759776287">rs759776287</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.066</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536621.6">MUC12:ENST00000536621.6:c.689A&gt;C:p.(Tyr230Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.06665444</dd>
                                    
                                    <dd>REVEL: 0.014</dd>
                                    <dd>MVP: 0.067</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0260%</dd>
                                    <dd>gnomAD_E_AMR: 0.0042%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KDM5A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000073614">KDM5A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.327</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.334 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KDM5A">KDM5A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002123, abnormal definitive hematopoiesis</dd><dd>HP:0001903, Anemia - MP:0002123, abnormal definitive hematopoiesis</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002123, abnormal definitive hematopoiesis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0000219, increased neutrophil cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KDM5A%0DARID4A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARID4A</a> and phenotypic similarity 0.666 to mouse mutant of ARID4A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia - MP:0001577, anemia</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002445, abnormal mononuclear cell differentiation</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0011747, myelofibrosis</dd><dd>HP:0001396, Cholestasis - MP:0000599, enlarged liver</dd><dd>HP:0001410, Decreased liver function - MP:0000599, enlarged liver</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000599, enlarged liver</dd><dd>HP:0001399, Hepatic failure - MP:0000599, enlarged liver</dd><dd>HP:0000938, Osteopenia - MP:0002397, abnormal bone marrow morphology</dd><dd>HP:0001653, Mitral regurgitation - MP:0004834, ovary hemorrhage</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000599, enlarged liver</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000599, enlarged liver</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000599, enlarged liver</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000223, decreased monocyte cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.327</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>INS</b>
                            <b>12-323210-C-CA [0/1:./.:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs60377454">rs60377454</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399788.7">KDM5A:ENST00000399788.7:c.2151-5_2151-4insT:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LRRK2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188906">LRRK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.291</b>
                    <data>(p=1.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.512</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.771</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.297 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=411602">Hereditary late-onset Parkinson disease</a> associated with LRRK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0002171, Gliosis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0004926, Orthostatic hypotension due to autonomic dysfunction</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0002171, Gliosis</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001824, Weight loss</dd></dl><dl><dt>Phenotypic similarity 0.277 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LRRK2">LRRK2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia - MP:0008918, microgliosis</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008918, microgliosis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0008918, microgliosis</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0008918, microgliosis</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0008918, microgliosis</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0008918, microgliosis</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.251 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=LRRK2">LRRK2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - ZP:0019952, otic vesicle edematous, abnormal</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - ZP:0019959, spinal cord has fewer parts of type neuron, abnormal</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRK2%0DLRRK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRRK1</a> and phenotypic similarity 0.418 to <a href="http://www.omim.org/entry/615198">Osteosclerotic metaphyseal dysplasia</a> associated with LRRK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0011001, Increased bone mineral density</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia - HP:0011001, Increased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LRRK2%0DLRRK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRRK1</a> and phenotypic similarity 0.630 to mouse mutant of LRRK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0005012, decreased eosinophil cell number</dd><dd>HP:0001903, Anemia - MP:0010067, increased red blood cell distribution width</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0005012, decreased eosinophil cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0000221, decreased leukocyte cell number</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function - MP:0002966, decreased circulating alkaline phosphatase level</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002966, decreased circulating alkaline phosphatase level</dd><dd>HP:0001399, Hepatic failure - MP:0002966, decreased circulating alkaline phosphatase level</dd><dd>HP:0000938, Osteopenia - MP:0000062, increased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005343, increased circulating aspartate transaminase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0010090, increased circulating creatine kinase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0001556, increased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - MP:0005343, increased circulating aspartate transaminase level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002966, decreased circulating alkaline phosphatase level</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002966, decreased circulating alkaline phosphatase level</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0002764, short tibia</dd></dl><dl><dt>PhenIX semantic similarity score: 0.46 (p-value: 0.605260)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/607060">OMIM:607060</a> Parkinson disease 8 (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2828">ORPHA:2828</a> Young-onset Parkinson disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=411602">ORPHA:411602</a> Hereditary late-onset Parkinson disease - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.291</b>
                                    <data>(p=1.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.512</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.771</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.158 to ORPHA:2828 Young-onset Parkinson disease</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>12-40315306-T-A [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs550610903">rs550610903</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=787873%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.787</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298910.12">LRRK2:ENST00000298910.12:c.4827+6T&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0156%</dd>
                                    <dd>TOPMed: 0.0156%</dd>
                                    <dd>ExAC EAS: 0.0809%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_EAS: 0.1102%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_EAS: 0.0621%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-40295498-A-G [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs573263484">rs573263484</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.754</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298910.12">LRRK2:ENST00000298910.12:c.2950A&gt;G:p.(Ile984Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.75450885</dd>
                                    
                                    <dd>REVEL: 0.135</dd>
                                    <dd>MVP: 0.755</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPDYE5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170092">SPDYE5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.249</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.289</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.289 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SPDYE5">SPDYE5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000689, abnormal spleen morphology</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000689, abnormal spleen morphology</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0000689, abnormal spleen morphology</dd><dd>HP:0001410, Decreased liver function - MP:0000689, abnormal spleen morphology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000689, abnormal spleen morphology</dd><dd>HP:0001399, Hepatic failure - MP:0000689, abnormal spleen morphology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000689, abnormal spleen morphology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000689, abnormal spleen morphology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000689, abnormal spleen morphology</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.249</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.289</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                            <b>DEL</b>
                            <b>7-75497938-AG-A [0/1:0/0:0/0]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000625065.4">SPDYE5:ENST00000625065.4:c.613del:p.(Asp205Ilefs*14)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MYO7A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137474">MYO7A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.248</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.747</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.430 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MYO7A">MYO7A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0010123, increased bone mineral content</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia - MP:0000063, decreased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>Phenotypic similarity 0.169 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=MYO7A">MYO7A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - ZP:0010818, inner ear receptor cell stereocilium organization disrupted, abnormal</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYO7A%0DWHRN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to WHRN</a> and phenotypic similarity 0.607 to mouse mutant of WHRN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000221, decreased leukocyte cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0000221, decreased leukocyte cell number</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0010398, decreased liver glycogen level</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0010398, decreased liver glycogen level</dd><dd>HP:0001399, Hepatic failure - MP:0010398, decreased liver glycogen level</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0001745, increased circulating corticosterone level</dd><dd>HP:0001873, Thrombocytopenia - MP:0000221, decreased leukocyte cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0010398, decreased liver glycogen level</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0010398, decreased liver glycogen level</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0004362, cochlear hair cell degeneration</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0010398, decreased liver glycogen level</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000221, decreased leukocyte cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>PhenIX semantic similarity score: 0.08 (p-value: 0.999990)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/276900">OMIM:276900</a> Usher syndrome, type 1B - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/600060">OMIM:600060</a> Deafness, autosomal recessive 2 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/601317">OMIM:601317</a> Deafness, autosomal dominant 11 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=231169">ORPHA:231169</a> Usher syndrome type 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=231178">ORPHA:231178</a> Usher syndrome type 2 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.248</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.747</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-77199723-A-G [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201251963">rs201251963</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=194096%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.755</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409709.9">MYO7A:ENST00000409709.9:c.4757A&gt;G:p.(Asn1586Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.82669157</dd>
                                    
                                    <dd>REVEL: 0.320</dd>
                                    <dd>MVP: 0.827</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0468%</dd>
                                    <dd>TOPMed: 0.0336%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.1230%</dd>
                                    <dd>ESP All: 0.0401%</dd>
                                    <dd>ExAC AFR: 0.0432%</dd>
                                    <dd>ExAC AMR: 0.0176%</dd>
                                    <dd>ExAC EAS: 0.4086%</dd>
                                    <dd>ExAC OTH: 0.2326%</dd>
                                    <dd>ExAC SAS: 0.0063%</dd>
                                    <dd>gnomAD_E_AFR: 0.0527%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                    <dd>gnomAD_E_EAS: 0.4931%</dd>
                                    <dd>gnomAD_E_OTH: 0.1098%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                    <dd>gnomAD_G_AFR: 0.1032%</dd>
                                    <dd>gnomAD_G_EAS: 0.4321%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-77204123-G-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369424114">rs369424114</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=738306%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.739</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409709.9">MYO7A:ENST00000409709.9:c.5374G&gt;A:p.(Val1792Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.78195363</dd>
                                    
                                    <dd>REVEL: 0.254</dd>
                                    <dd>MVP: 0.782</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0937%</dd>
                                    <dd>TOPMed: 0.0042%</dd>
                                    <dd>ESP EA: 0.0119%</dd>
                                    <dd>ESP All: 0.0080%</dd>
                                    <dd>ExAC AFR: 0.0233%</dd>
                                    <dd>ExAC EAS: 0.0949%</dd>
                                    <dd>ExAC NFE: 0.0033%</dd>
                                    <dd>ExAC SAS: 0.3376%</dd>
                                    <dd>gnomAD_E_AFR: 0.0078%</dd>
                                    <dd>gnomAD_E_EAS: 0.0639%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_E_SAS: 0.2198%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CYP11B2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179142">CYP11B2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.133</b>
                    <data>(p=2.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.670</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.439 to <a href="http://www.omim.org/entry/610600">Hypoaldosteronism, congenital, due to CMO II deficiency</a> associated with CYP11B2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0020200, Increased circulating 18-hydroxycortisone level</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0001944, Dehydration</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0020200, Increased circulating 18-hydroxycortisone level</dd><dd>HP:0001410, Decreased liver function - HP:0004319, Decreased circulating aldosterone level</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0004319, Decreased circulating aldosterone level</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0001278, Orthostatic hypotension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000848, Increased circulating renin level</dd><dd>HP:0003073, Hypoalbuminemia - HP:0000848, Increased circulating renin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0004319, Decreased circulating aldosterone level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0000848, Increased circulating renin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0001278, Orthostatic hypotension</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0004319, Decreased circulating aldosterone level</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Phenotypic similarity 0.480 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CYP11B2">CYP11B2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production - MP:0004956, decreased thymus weight</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0002981, increased liver weight</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002981, increased liver weight</dd><dd>HP:0001399, Hepatic failure - MP:0002981, increased liver weight</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0006144, increased systemic arterial systolic blood pressure</dd><dd>HP:0200114, Metabolic alkalosis - MP:0001748, increased circulating adrenocorticotropin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0003353, decreased circulating renin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002665, decreased circulating corticosterone level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0001748, increased circulating adrenocorticotropin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0002176, increased brain weight</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0002981, increased liver weight</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP11B2%0DAKR1D1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AKR1D1</a> and phenotypic similarity 0.715 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79303">Congenital bile acid synthesis defect type 2</a> associated with AKR1D1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0002240, Hepatomegaly</dd><dd>HP:0001903, Anemia - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0001396, Cholestasis - HP:0001396, Cholestasis</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0002748, Rickets</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0030984, Abnormal serum bile acid concentration</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0030984, Abnormal serum bile acid concentration</dd><dd>HP:0001873, Thrombocytopenia - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0002151, Increased serum lactate - HP:0030984, Abnormal serum bile acid concentration</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001397, Hepatic steatosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP11B2%0DAKR1D1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to AKR1D1</a> and phenotypic similarity 0.161 to mouse mutant of AKR1D1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002833, increased heart weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0013278, decreased fasting circulating glucose level</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.92 (p-value: 0.093620)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/203400">OMIM:203400</a> Hypoaldosteronism, congenital, due to CMO I deficiency - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/610600">OMIM:610600</a> Hypoaldosteronism, congenital, due to CMO II deficiency - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=403">ORPHA:403</a> Familial hyperaldosteronism type I (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.133</b>
                                    <data>(p=2.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.670</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.439 to OMIM:610600 Hypoaldosteronism, congenital, due to CMO II deficiency</dt>
                                
                                    <dt>Phenotypic similarity 0.431 to OMIM:203400 Hypoaldosteronism, congenital, due to CMO I deficiency</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-142917102-C-T [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs372556807">rs372556807</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=313362%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.670</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323110.2">CYP11B2:ENST00000323110.2:c.352G&gt;A:p.(Ala118Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.67928296</dd>
                                    
                                    <dd>REVEL: 0.285</dd>
                                    <dd>MVP: 0.679</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0312%</dd>
                                    <dd>TOPMed: 0.0034%</dd>
                                    <dd>ExAC EAS: 0.0925%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_EAS: 0.0754%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0097%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NPHS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161270">NPHS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.065</b>
                    <data>(p=3.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.570</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.511</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.570 to <a href="http://www.omim.org/entry/256300">Nephrotic syndrome, type 1</a> associated with NPHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0003270, Abdominal distention</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0000969, Edema</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0000821, Hypothyroidism</dd><dd>HP:0001410, Decreased liver function - HP:0000821, Hypothyroidism</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0000821, Hypothyroidism</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000969, Edema</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003073, Hypoalbuminemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003077, Hyperlipidemia</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0003075, Hypoproteinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000821, Hypothyroidism</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001518, Small for gestational age</dd></dl><dl><dt>Phenotypic similarity 0.313 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NPHS1">NPHS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0003068, enlarged kidney</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0001785, edema</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002962, increased urine protein level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002962, increased urine protein level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.219 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=NPHS1">NPHS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - ZP:0010821, yolk syncytial layer edematous, abnormal</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0010898, pronephric glomerular capillary hypotrophic, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPHS1%0DCD2AP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CD2AP</a> and phenotypic similarity 0.529 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=656">Genetic steroid-resistant nephrotic syndrome</a> associated with CD2AP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0002586, Peritonitis</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0000969, Edema</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0002586, Peritonitis</dd><dd>HP:0001410, Decreased liver function - HP:0002586, Peritonitis</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0002586, Peritonitis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000969, Edema</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003073, Hypoalbuminemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003073, Hypoalbuminemia</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0003073, Hypoalbuminemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0002586, Peritonitis</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPHS1%0DCD2AP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CD2AP</a> and phenotypic similarity 0.668 to mouse mutant of CD2AP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0003643, spleen atrophy</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0005324, ascites</dd><dd>HP:0010972, Anemia of inadequate production - MP:0003643, spleen atrophy</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0003644, thymus atrophy</dd><dd>HP:0001410, Decreased liver function - MP:0003644, thymus atrophy</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0003644, thymus atrophy</dd><dd>HP:0001399, Hepatic failure - MP:0003644, thymus atrophy</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005553, increased circulating creatinine level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0005419, decreased circulating serum albumin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0005419, decreased circulating serum albumin level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0005553, increased circulating creatinine level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0003644, thymus atrophy</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0003644, thymus atrophy</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0003644, thymus atrophy</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0005264, glomerulosclerosis</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPHS1%0DCD2AP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CD2AP</a> and phenotypic similarity 0.210 to fish mutant of CD2AP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - ZP:0002202, whole organism edematous, abnormal</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0018769, pronephric glomerular capillary distended, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy - ZP:0002202, whole organism edematous, abnormal</dd></dl><dl><dt>PhenIX semantic similarity score: 0.84 (p-value: 0.016680)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/256300">OMIM:256300</a> Nephrotic syndrome, type 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=656">ORPHA:656</a> Genetic steroid-resistant nephrotic syndrome - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=839">ORPHA:839</a> Congenital nephrotic syndrome, Finnish type - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.065</b>
                                    <data>(p=3.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.570</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.511</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.570 to OMIM:256300 Nephrotic syndrome, type 1</dt>
                                
                                    <dt>Phenotypic similarity 0.529 to ORPHA:656 Genetic steroid-resistant nephrotic syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.291 to ORPHA:839 Congenital nephrotic syndrome, Finnish type</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>19-35844248-G-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1973103431">rs1973103431</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1148728%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378910.10">NPHS1:ENST00000378910.10:c.2072-5C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-35849273-C-T [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs115308424">rs115308424</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=741816%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.222</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378910.10">NPHS1:ENST00000378910.10:c.803G&gt;A:p.(Arg268Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.35008886</dd>
                                    
                                    <dd>REVEL: 0.132</dd>
                                    <dd>MVP: 0.350</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6402%</dd>
                                    <dd>TOPMed: 0.0771%</dd>
                                    <dd>ESP AA: 0.0681%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0405%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 1.2050%</dd>
                                    <dd>ExAC NFE: 0.0046%</dd>
                                    <dd>ExAC OTH: 0.3401%</dd>
                                    <dd>ExAC SAS: 0.2926%</dd>
                                    <dd>gnomAD_E_AFR: 0.0332%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_EAS: 1.3077%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0916%</dd>
                                    <dd>gnomAD_E_SAS: 0.2665%</dd>
                                    <dd>gnomAD_G_AFR: 0.0574%</dd>
                                    <dd>gnomAD_G_EAS: 0.9889%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>19-35851580-G-A [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs114385015">rs114385015</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    <div class="col-sm-12">
                        <b>ClinVar: </b>
                        
                        
                            
                            
                            
                            
                            
                            <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                            <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=348619%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                        
                    </div>
                    <div class="col-sm-6">
                        <b>Variant score: 0.074</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378910.10">NPHS1:ENST00000378910.10:c.151C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.6402%</dd>
                            <dd>TOPMed: 0.0801%</dd>
                            <dd>ESP AA: 0.0908%</dd>
                            <dd>ESP EA: 0.0349%</dd>
                            <dd>ESP All: 0.0539%</dd>
                            <dd>ExAC AFR: 0.0218%</dd>
                            <dd>ExAC AMR: 0.0267%</dd>
                            <dd>ExAC EAS: 1.0331%</dd>
                            <dd>ExAC NFE: 0.0158%</dd>
                            <dd>ExAC OTH: 0.3606%</dd>
                            <dd>ExAC SAS: 0.2790%</dd>
                            <dd>gnomAD_E_AFR: 0.0200%</dd>
                            <dd>gnomAD_E_AMR: 0.0388%</dd>
                            <dd>gnomAD_E_EAS: 1.0627%</dd>
                            <dd>gnomAD_E_NFE: 0.0185%</dd>
                            <dd>gnomAD_E_OTH: 0.0920%</dd>
                            <dd>gnomAD_E_SAS: 0.2601%</dd>
                            <dd>gnomAD_G_AFR: 0.0688%</dd>
                            <dd>gnomAD_G_EAS: 0.7398%</dd>
                            <dd>gnomAD_G_NFE: 0.0133%</dd>
                            <dd>gnomAD_G_OTH: 0.1020%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT6B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185479">KRT6B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.039</b>
                    <data>(p=3.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.181</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.895</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.337 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2309">Pachyonychia congenita</a> associated with KRT6B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0007490, Linear arrays of macular hyperkeratoses in flexural areas</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Phenotypic similarity 0.361 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KRT6B">KRT6B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>PhenIX semantic similarity score: 0.64 (p-value: 0.157600)</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615728">OMIM:615728</a> Pachyonychia congenita 4 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2309">ORPHA:2309</a> Pachyonychia congenita - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.039</b>
                                    <data>(p=3.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.181</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.895</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-52449543-C-T [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369853484">rs369853484</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.895</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252252.4">KRT6B:ENST00000252252.4:c.1003G&gt;A:p.(Ala335Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9026787</dd>
                                    
                                    <dd>REVEL: 0.420</dd>
                                    <dd>MVP: 0.903</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC SAS: 0.0606%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_SAS: 0.0552%</dd>
                                    <dd>gnomAD_G_EAS: 0.0619%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="URB2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135763">URB2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.039</b>
                    <data>(p=3.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.531</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.301 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=URB2">URB2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0001903, Anemia - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0001396, Cholestasis - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0001410, Decreased liver function - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0002910, Elevated hepatic transaminase - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0001399, Hepatic failure - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0018768, thymus absent, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB2%0DIRS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRS1</a> and phenotypic similarity 0.258 to <a href="http://www.omim.org/entry/125853">Type 2 diabetes mellitus, susceptibility to</a> associated with IRS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0001410, Decreased liver function - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000855, Insulin resistance</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0000855, Insulin resistance</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0031819, Increased waist to hip ratio</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB2%0DIRS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRS1</a> and phenotypic similarity 0.638 to mouse mutant of IRS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0001903, Anemia - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0004989, decreased osteoblast cell number</dd><dd>HP:0001396, Cholestasis - MP:0003059, decreased insulin secretion</dd><dd>HP:0001410, Decreased liver function - MP:0003059, decreased insulin secretion</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0003059, decreased insulin secretion</dd><dd>HP:0001399, Hepatic failure - MP:0003059, decreased insulin secretion</dd><dd>HP:0000938, Osteopenia - MP:0000063, decreased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation - MP:0006143, increased systemic arterial diastolic blood pressure</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002079, increased circulating insulin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002079, increased circulating insulin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0002151, Increased serum lactate - MP:0001554, increased circulating free fatty acids level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0003059, decreased insulin secretion</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0003059, decreased insulin secretion</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0003059, decreased insulin secretion</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=URB2%0DIRS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRS1</a> and phenotypic similarity 0.299 to fish mutant of IRS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - ZP:0005798, developmental growth decreased rate, abnormal</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.039</b>
                                    <data>(p=3.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.531</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-229635178-A-C [0/1:0/1:0/0]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.613</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258243.7">URB2:ENST00000258243.7:c.565A&gt;C:p.(Asn189His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.61290497</dd>
                                    
                                    <dd>REVEL: 0.365</dd>
                                    <dd>MVP: 0.613</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-229647556-T-C [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs371627284">rs371627284</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.449</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258243.7">URB2:ENST00000258243.7:c.3953T&gt;C:p.(Val1318Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4515046</dd>
                                    
                                    <dd>REVEL: 0.110</dd>
                                    <dd>MVP: 0.452</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0156%</dd>
                                    <dd>TOPMed: 0.0079%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC SAS: 0.0242%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0232%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR10G9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000236981">OR10G9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.034</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.517</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR10G9%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.592 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=342">Familial Mediterranean fever</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0002716, Lymphadenopathy</dd><dd>HP:0001541, Ascites - HP:0001541, Ascites</dd><dd>HP:0010972, Anemia of inadequate production - HP:0002716, Lymphadenopathy</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0002716, Lymphadenopathy</dd><dd>HP:0001396, Cholestasis - HP:0006554, Acute hepatic failure</dd><dd>HP:0001410, Decreased liver function - HP:0006554, Acute hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0006554, Acute hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0001369, Arthritis</dd><dd>HP:0001653, Mitral regurgitation - HP:0001658, Myocardial infarction</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001945, Fever</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0002716, Lymphadenopathy</dd><dd>HP:0002151, Increased serum lactate - HP:0001945, Fever</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0006554, Acute hepatic failure</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0002633, Vasculitis</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR10G9%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.615 to mouse mutant of MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000691, enlarged spleen</dd><dd>HP:0001903, Anemia - MP:0001577, anemia</dd><dd>HP:0001541, Ascites - MP:0000691, enlarged spleen</dd><dd>HP:0010972, Anemia of inadequate production - MP:0001577, anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0000218, increased leukocyte cell number</dd><dd>HP:0001396, Cholestasis - MP:0001860, liver inflammation</dd><dd>HP:0001410, Decreased liver function - MP:0001860, liver inflammation</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0001860, liver inflammation</dd><dd>HP:0001399, Hepatic failure - MP:0001860, liver inflammation</dd><dd>HP:0000938, Osteopenia - MP:0002933, joint inflammation</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0001873, Thrombocytopenia - MP:0000322, increased granulocyte number</dd><dd>HP:0002151, Increased serum lactate - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0001860, liver inflammation</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0001860, liver inflammation</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0001860, liver inflammation</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0005425, increased macrophage cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.034</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.517</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-124023707-G-C [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs767983467">rs767983467</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.720</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.695G&gt;C:p.(Gly232Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7209725</dd>
                                    
                                    <dd>REVEL: 0.126</dd>
                                    <dd>MVP: 0.721</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-124023083-C-G [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs113347250">rs113347250</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.313</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.71C&gt;G:p.(Pro24Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.44500792</dd>
                                    
                                    <dd>REVEL: 0.076</dd>
                                    <dd>MVP: 0.445</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4841%</dd>
                                    <dd>TOPMed: 0.3234%</dd>
                                    <dd>ESP AA: 1.1586%</dd>
                                    <dd>ESP All: 0.3923%</dd>
                                    <dd>ExAC AFR: 1.0475%</dd>
                                    <dd>ExAC AMR: 0.0518%</dd>
                                    <dd>ExAC SAS: 0.1757%</dd>
                                    <dd>gnomAD_E_AFR: 0.9344%</dd>
                                    <dd>gnomAD_E_AMR: 0.0744%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.1819%</dd>
                                    <dd>gnomAD_G_AFR: 0.8722%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>11-124023660-T-C [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs141129525">rs141129525</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.096</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.648T&gt;C:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.1093%</dd>
                            <dd>TOPMed: 0.0672%</dd>
                            <dd>ESP AA: 0.0909%</dd>
                            <dd>ESP All: 0.0308%</dd>
                            <dd>ExAC AFR: 0.1345%</dd>
                            <dd>ExAC AMR: 0.0518%</dd>
                            <dd>ExAC EAS: 0.0578%</dd>
                            <dd>ExAC NFE: 0.0150%</dd>
                            <dd>gnomAD_E_AFR: 0.1438%</dd>
                            <dd>gnomAD_E_AMR: 0.0387%</dd>
                            <dd>gnomAD_E_EAS: 0.0522%</dd>
                            <dd>gnomAD_E_FIN: 0.0045%</dd>
                            <dd>gnomAD_E_NFE: 0.0099%</dd>
                            <dd>gnomAD_E_SAS: 0.0032%</dd>
                            <dd>gnomAD_G_AFR: 0.2522%</dd>
                            <dd>gnomAD_G_EAS: 0.1233%</dd>
                            <dd>gnomAD_G_NFE: 0.0067%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>11-124023543-G-C [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs200638744">rs200638744</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.095</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.531G&gt;C:p.(Leu177Phe)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.13424169</dd>
                            
                            <dd>REVEL: 0.076</dd>
                            <dd>MVP: 0.134</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.4060%</dd>
                            <dd>gnomAD_E_AFR: 0.0131%</dd>
                            <dd>gnomAD_E_AMR: 0.0417%</dd>
                            <dd>gnomAD_E_EAS: 0.0232%</dd>
                            <dd>gnomAD_E_FIN: 0.0090%</dd>
                            <dd>gnomAD_E_NFE: 0.0090%</dd>
                            <dd>gnomAD_E_OTH: 0.1463%</dd>
                            <dd>gnomAD_E_SAS: 1.1439%</dd>
                            <dd>gnomAD_G_AFR: 0.0229%</dd>
                            <dd>gnomAD_G_FIN: 0.0286%</dd>
                            <dd>gnomAD_G_NFE: 0.0267%</dd>
                            <dd>gnomAD_G_OTH: 0.3055%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>11-124023210-C-T [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs112152472">rs112152472</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.080</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.198C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.4841%</dd>
                            <dd>TOPMed: 0.3230%</dd>
                            <dd>ESP AA: 0.8636%</dd>
                            <dd>ESP All: 0.2933%</dd>
                            <dd>gnomAD_E_AFR: 0.9017%</dd>
                            <dd>gnomAD_E_AMR: 0.0747%</dd>
                            <dd>gnomAD_E_OTH: 0.0369%</dd>
                            <dd>gnomAD_E_SAS: 0.1760%</dd>
                            <dd>gnomAD_G_AFR: 0.8601%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>11-124023540-T-C [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs569353053">rs569353053</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.068</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.528T&gt;C:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.4060%</dd>
                            <dd>gnomAD_E_AFR: 0.0065%</dd>
                            <dd>gnomAD_E_AMR: 0.0387%</dd>
                            <dd>gnomAD_E_EAS: 0.0232%</dd>
                            <dd>gnomAD_E_FIN: 0.0090%</dd>
                            <dd>gnomAD_E_NFE: 0.0081%</dd>
                            <dd>gnomAD_E_OTH: 0.1645%</dd>
                            <dd>gnomAD_E_SAS: 1.2120%</dd>
                            <dd>gnomAD_G_AFR: 0.0229%</dd>
                            <dd>gnomAD_G_FIN: 0.0286%</dd>
                            <dd>gnomAD_G_NFE: 0.0267%</dd>
                            <dd>gnomAD_G_OTH: 0.3055%</dd>
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>11-124023549-T-C [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs2924497">rs2924497</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.050</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375024.1">OR10G9:ENST00000375024.1:c.537T&gt;C:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.4528%</dd>
                            <dd>TOPMed: 0.0884%</dd>
                            <dd>gnomAD_E_AFR: 0.0327%</dd>
                            <dd>gnomAD_E_AMR: 0.0834%</dd>
                            <dd>gnomAD_E_EAS: 0.0232%</dd>
                            <dd>gnomAD_E_FIN: 0.0224%</dd>
                            <dd>gnomAD_E_NFE: 0.0772%</dd>
                            <dd>gnomAD_E_OTH: 0.1828%</dd>
                            <dd>gnomAD_E_SAS: 1.5457%</dd>
                            <dd>gnomAD_G_AFR: 0.0459%</dd>
                            <dd>gnomAD_G_AMR: 0.1196%</dd>
                            <dd>gnomAD_G_EAS: 0.0617%</dd>
                            <dd>gnomAD_G_FIN: 0.0572%</dd>
                            <dd>gnomAD_G_NFE: 0.0533%</dd>
                            <dd>gnomAD_G_OTH: 0.4073%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172867">KRT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.034</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.799</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.250 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=455">Superficial epidermolytic ichthyosis</a> associated with KRT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0000969, Edema</dd><dd>HP:0010972, Anemia of inadequate production - HP:0008064, Ichthyosis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000969, Edema</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0000969, Edema</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.267 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KRT2">KRT2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0002796, impaired skin barrier function</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002656, abnormal keratinocyte differentiation</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT2%0DCYP27A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CYP27A1</a> and phenotypic similarity 0.613 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=909">Cerebrotendinous xanthomatosis</a> associated with CYP27A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001081, Cholelithiasis</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0002171, Gliosis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0002659, Increased susceptibility to fractures</dd><dd>HP:0001396, Cholestasis - HP:0006579, Prolonged neonatal jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0000821, Hypothyroidism</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0012379, Abnormal circulating enzyme concentration or activity</dd><dd>HP:0001399, Hepatic failure - HP:0000821, Hypothyroidism</dd><dd>HP:0000938, Osteopenia - HP:0000938, Osteopenia</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0002151, Increased serum lactate</dd><dd>HP:0003073, Hypoalbuminemia - HP:0002151, Increased serum lactate</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0002151, Increased serum lactate</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0002151, Increased serum lactate</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0012379, Abnormal circulating enzyme concentration or activity</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0012379, Abnormal circulating enzyme concentration or activity</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0002196, Myelopathy</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001081, Cholelithiasis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0004416, Precocious atherosclerosis</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT2%0DCYP27A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CYP27A1</a> and phenotypic similarity 0.632 to mouse mutant of CYP27A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia - MP:0008935, decreased mean platelet volume</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008935, decreased mean platelet volume</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0002981, increased liver weight</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002981, increased liver weight</dd><dd>HP:0001399, Hepatic failure - MP:0002981, increased liver weight</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0010393, shortened QRS complex duration</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0008935, decreased mean platelet volume</dd><dd>HP:0002151, Increased serum lactate - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0002981, increased liver weight</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0008935, decreased mean platelet volume</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.17 (p-value: 0.683470)</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/146800">OMIM:146800</a> Ichthyosis bullosa of Siemens - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=455">ORPHA:455</a> Superficial epidermolytic ichthyosis - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.034</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.799</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>12-52647860-T-C [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1354217118">rs1354217118</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.799</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309680.4">KRT2:ENST00000309680.4:c.1123-5A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CIP2A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163507">CIP2A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.033</b>
                    <data>(p=3.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.512</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CIP2A%0DPOU2AF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POU2AF1</a> and phenotypic similarity 0.679 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=186">Primary biliary cholangitis</a> associated with POU2AF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001409, Portal hypertension</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0001541, Ascites</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0000939, Osteoporosis</dd><dd>HP:0001396, Cholestasis - HP:0000952, Jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0000939, Osteoporosis</dd><dd>HP:0001653, Mitral regurgitation - HP:0001278, Orthostatic hypotension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0002908, Conjugated hyperbilirubinemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003073, Hypoalbuminemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003119, Abnormal circulating lipid concentration</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0002908, Conjugated hyperbilirubinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0003155, Elevated circulating alkaline phosphatase concentration</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0001278, Orthostatic hypotension</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001395, Hepatic fibrosis</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CIP2A%0DPOU2AF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POU2AF1</a> and phenotypic similarity 0.389 to mouse mutant of POU2AF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0008474, absent spleen germinal center</dd><dd>HP:0001903, Anemia - MP:0002144, abnormal B cell differentiation</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002144, abnormal B cell differentiation</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0002144, abnormal B cell differentiation</dd><dd>HP:0001396, Cholestasis - MP:0008474, absent spleen germinal center</dd><dd>HP:0001410, Decreased liver function - MP:0008474, absent spleen germinal center</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0008474, absent spleen germinal center</dd><dd>HP:0001399, Hepatic failure - MP:0008474, absent spleen germinal center</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0005017, decreased B cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0008474, absent spleen germinal center</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0008474, absent spleen germinal center</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0008474, absent spleen germinal center</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0002144, abnormal B cell differentiation</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.033</b>
                                    <data>(p=3.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.512</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-108560707-T-C [0/1:0/0:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.566</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295746.13">CIP2A:ENST00000295746.13:c.1769A&gt;G:p.(Asp590Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5658299</dd>
                                    
                                    <dd>REVEL: 0.029</dd>
                                    <dd>MVP: 0.566</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-108560731-T-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147916833">rs147916833</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.457</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295746.13">CIP2A:ENST00000295746.13:c.1745A&gt;T:p.(Lys582Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5657933</dd>
                                    
                                    <dd>REVEL: 0.082</dd>
                                    <dd>MVP: 0.566</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1093%</dd>
                                    <dd>TOPMed: 0.1093%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC EAS: 0.7518%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_EAS: 0.8827%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_EAS: 0.7481%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RAB3A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105649">RAB3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.023</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.521</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.450</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAB3A%0DRAB3GAP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP2</a> and phenotypic similarity 0.245 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2510">Micro syndrome</a> associated with RAB3GAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0000823, Delayed puberty</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001339, Lissencephaly</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0000823, Delayed puberty</dd><dd>HP:0001410, Decreased liver function - HP:0000823, Delayed puberty</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0000823, Delayed puberty</dd><dd>HP:0000938, Osteopenia - HP:0001387, Joint stiffness</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0009830, Peripheral neuropathy</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000823, Delayed puberty</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAB3A%0DRAB3GAP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB3GAP2</a> and phenotypic similarity 0.602 to mouse mutant of RAB3GAP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia - MP:0000208, decreased hematocrit</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000208, decreased hematocrit</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0001399, Hepatic failure - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0005333, decreased heart rate</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0008259, abnormal optic disk morphology</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0003179, thrombocytopenia</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.023</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.521</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.450</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>19-18198852-A-G [0/1:0/1:0/0]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000222256.9">RAB3A:ENST00000222256.9:c.348-3T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-18197593-C-T [0/1:0/0:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000222256.9">RAB3A:ENST00000222256.9:c.540G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TIAM1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000156299">TIAM1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.023</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.465</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.286 to <a href="http://www.omim.org/entry/619908">Neurodevelopmental disorder with language delay and seizures</a> associated with TIAM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0000821, Hypothyroidism</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0003429, CNS hypomyelination</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0000821, Hypothyroidism</dd><dd>HP:0001410, Decreased liver function - HP:0000821, Hypothyroidism</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0000821, Hypothyroidism</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - HP:0002917, Hypomagnesemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0002917, Hypomagnesemia</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0002917, Hypomagnesemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0003429, CNS hypomyelination</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000821, Hypothyroidism</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.192 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TIAM1">TIAM1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0000783, abnormal forebrain morphology</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TIAM1%0DANK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANK1</a> and phenotypic similarity 0.543 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=822">Hereditary spherocytosis</a> associated with ANK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0001903, Anemia</dd><dd>HP:0001541, Ascites - HP:0003270, Abdominal distention</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001978, Extramedullary hematopoiesis</dd><dd>HP:0001396, Cholestasis - HP:0000952, Jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0001081, Cholelithiasis</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001081, Cholelithiasis</dd><dd>HP:0000938, Osteopenia - HP:0001997, Gout</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001945, Fever</dd><dd>HP:0003073, Hypoalbuminemia - HP:0002904, Hyperbilirubinemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0002904, Hyperbilirubinemia</dd><dd>HP:0001873, Thrombocytopenia - HP:0004444, Spherocytosis</dd><dd>HP:0002151, Increased serum lactate - HP:0002904, Hyperbilirubinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0002240, Hepatomegaly</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001903, Anemia</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TIAM1%0DANK1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANK1</a> and phenotypic similarity 0.758 to mouse mutant of ANK1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008387, hypochromic anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0002397, abnormal bone marrow morphology</dd><dd>HP:0001396, Cholestasis - MP:0000611, jaundice</dd><dd>HP:0001410, Decreased liver function - MP:0000611, jaundice</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000611, jaundice</dd><dd>HP:0001399, Hepatic failure - MP:0000611, jaundice</dd><dd>HP:0000938, Osteopenia - MP:0002397, abnormal bone marrow morphology</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005637, abnormal iron homeostasis</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002874, decreased hemoglobin content</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002874, decreased hemoglobin content</dd><dd>HP:0001873, Thrombocytopenia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0002151, Increased serum lactate - MP:0005637, abnormal iron homeostasis</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000611, jaundice</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000611, jaundice</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0000876, Purkinje cell degeneration</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000599, enlarged liver</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000218, increased leukocyte cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619908">OMIM:619908</a> Neurodevelopmental disorder with language delay and seizures - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.023</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.465</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.286 to OMIM:619908 Neurodevelopmental disorder with language delay and seizures</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>21-31135955-C-G [0/1:0/0:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.583</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541036.6">TIAM1:ENST00000541036.6:c.3861G&gt;C:p.(Lys1287Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.583064</dd>
                                    
                                    <dd>REVEL: 0.165</dd>
                                    <dd>MVP: 0.583</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>21-31165060-T-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs373619112">rs373619112</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.348</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541036.6">TIAM1:ENST00000541036.6:c.2893A&gt;T:p.(Ser965Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.35421845</dd>
                                    
                                    <dd>REVEL: 0.061</dd>
                                    <dd>MVP: 0.354</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1093%</dd>
                                    <dd>TOPMed: 0.0155%</dd>
                                    <dd>ExAC AFR: 0.0290%</dd>
                                    <dd>ExAC EAS: 0.1276%</dd>
                                    <dd>ExAC SAS: 0.0366%</dd>
                                    <dd>gnomAD_E_AFR: 0.0392%</dd>
                                    <dd>gnomAD_E_EAS: 0.1102%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0357%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.1233%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT72" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170486">KRT72</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.019</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.394</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.571</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.394 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KRT72">KRT72</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000689, abnormal spleen morphology</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000689, abnormal spleen morphology</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0000598, abnormal liver morphology</dd><dd>HP:0001410, Decreased liver function - MP:0000598, abnormal liver morphology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000598, abnormal liver morphology</dd><dd>HP:0001399, Hepatic failure - MP:0000598, abnormal liver morphology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000598, abnormal liver morphology</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.019</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.394</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.571</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-52591508-C-T [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs747007532">rs747007532</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.571</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000293745.7">KRT72:ENST00000293745.7:c.919G&gt;A:p.(Ala307Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5721736</dd>
                                    
                                    <dd>REVEL: 0.365</dd>
                                    <dd>MVP: 0.572</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CT45A6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278289">CT45A6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.018</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.364</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.600</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.364 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CT45A6">CT45A6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0001556, increased circulating HDL cholesterol level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0001556, increased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.018</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.364</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.600</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-135798830-T-G [1/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1556582560">rs1556582560</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.600</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620654.2">CT45A6:ENST00000620654.2:c.164A&gt;C:p.(Glu55Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.018</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.364</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.600</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-135798830-T-G [1/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1556582560">rs1556582560</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.600</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000620654.2">CT45A6:ENST00000620654.2:c.164A&gt;C:p.(Glu55Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAH5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000039139">DNAH5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.422</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.327 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001746, Asplenia</dd><dd>HP:0001903, Anemia - HP:0001746, Asplenia</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001746, Asplenia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001746, Asplenia</dd><dd>HP:0001396, Cholestasis - HP:0001746, Asplenia</dd><dd>HP:0001410, Decreased liver function - HP:0001746, Asplenia</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001746, Asplenia</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0001746, Asplenia</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001746, Asplenia</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001669, Transposition of the great arteries</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.418 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DNAH5">DNAH5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000689, abnormal spleen morphology</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000689, abnormal spleen morphology</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0000598, abnormal liver morphology</dd><dd>HP:0001410, Decreased liver function - MP:0000598, abnormal liver morphology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000598, abnormal liver morphology</dd><dd>HP:0001399, Hepatic failure - MP:0000598, abnormal liver morphology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0001891, hydrocephaly</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000598, abnormal liver morphology</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0004113, abnormal aortic arch morphology</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH5%0DALMS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALMS1</a> and phenotypic similarity 0.694 to <a href="http://www.omim.org/entry/203800">Alstrom syndrome</a> associated with ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0002240, Hepatomegaly</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0000956, Acanthosis nigricans</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0002240, Hepatomegaly</dd><dd>HP:0001410, Decreased liver function - HP:0000821, Hypothyroidism</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0000821, Hypothyroidism</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0000822, Hypertension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000842, Hyperinsulinemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0000842, Hyperinsulinemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003233, Decreased HDL cholesterol concentration</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0000842, Hyperinsulinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001397, Hepatic steatosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0002621, Atherosclerosis</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001956, Truncal obesity</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH5%0DALMS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALMS1</a> and phenotypic similarity 0.518 to mouse mutant of ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0031115, macrovesicular hepatic steatosis</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0031115, macrovesicular hepatic steatosis</dd><dd>HP:0001399, Hepatic failure - MP:0031115, macrovesicular hepatic steatosis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002079, increased circulating insulin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002079, increased circulating insulin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0005178, increased circulating cholesterol level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0002079, increased circulating insulin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0031115, macrovesicular hepatic steatosis</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0031115, macrovesicular hepatic steatosis</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0011967, increased or absent threshold for auditory brainstem response</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0031115, macrovesicular hepatic steatosis</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001260, increased body weight</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH5%0DALMS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALMS1</a> and phenotypic similarity 0.242 to fish mutant of ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - ZP:0001192, heart edematous, abnormal</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0001410, Decreased liver function - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0002910, Elevated hepatic transaminase - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0001399, Hepatic failure - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - ZP:0012993, endocrine pancreas morphology, abnormal</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.15 (p-value: 0.605320)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/608644">OMIM:608644</a> Ciliary dyskinesia, primary, 3, with or without situs inversus - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.422</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.327 to ORPHA:244 Primary ciliary dyskinesia</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-13793636-C-G [0/1:0/0:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.744</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265104.5">DNAH5:ENST00000265104.5:c.8103G&gt;C:p.(Gln2701His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.74420047</dd>
                                    
                                    <dd>REVEL: 0.313</dd>
                                    <dd>MVP: 0.744</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-13922230-C-T [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs370805321">rs370805321</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=683707%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265104.5">DNAH5:ENST00000265104.5:c.537G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0117%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC EAS: 0.0583%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0756%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BTN2A2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124508">BTN2A2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.014</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.420</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BTN2A2%0DCYP27A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CYP27A1</a> and phenotypic similarity 0.613 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=909">Cerebrotendinous xanthomatosis</a> associated with CYP27A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001081, Cholelithiasis</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0002171, Gliosis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0002659, Increased susceptibility to fractures</dd><dd>HP:0001396, Cholestasis - HP:0006579, Prolonged neonatal jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0000821, Hypothyroidism</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0012379, Abnormal circulating enzyme concentration or activity</dd><dd>HP:0001399, Hepatic failure - HP:0000821, Hypothyroidism</dd><dd>HP:0000938, Osteopenia - HP:0000938, Osteopenia</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0002151, Increased serum lactate</dd><dd>HP:0003073, Hypoalbuminemia - HP:0002151, Increased serum lactate</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0002151, Increased serum lactate</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0002151, Increased serum lactate</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0012379, Abnormal circulating enzyme concentration or activity</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0012379, Abnormal circulating enzyme concentration or activity</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0002196, Myelopathy</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001081, Cholelithiasis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0004416, Precocious atherosclerosis</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BTN2A2%0DCYP27A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CYP27A1</a> and phenotypic similarity 0.632 to mouse mutant of CYP27A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia - MP:0008935, decreased mean platelet volume</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008935, decreased mean platelet volume</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0002981, increased liver weight</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002981, increased liver weight</dd><dd>HP:0001399, Hepatic failure - MP:0002981, increased liver weight</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0010393, shortened QRS complex duration</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0008935, decreased mean platelet volume</dd><dd>HP:0002151, Increased serum lactate - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0002981, increased liver weight</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0008935, decreased mean platelet volume</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.014</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.420</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-26392785-G-A [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs375693447">rs375693447</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.539</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356709.9">BTN2A2:ENST00000356709.9:c.1390G&gt;A:p.(Asp464Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.55155087</dd>
                                    
                                    <dd>REVEL: 0.238</dd>
                                    <dd>MVP: 0.552</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0053%</dd>
                                    <dd>ExAC EAS: 0.1502%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_EAS: 0.1391%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-26392596-C-G [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148562624">rs148562624</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.300</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356709.9">BTN2A2:ENST00000356709.9:c.1201C&gt;G:p.(His401Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3529108</dd>
                                    
                                    <dd>REVEL: 0.132</dd>
                                    <dd>MVP: 0.353</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0389%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0384%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC EAS: 0.3932%</dd>
                                    <dd>ExAC FIN: 0.0302%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC OTH: 0.2203%</dd>
                                    <dd>ExAC SAS: 0.0485%</dd>
                                    <dd>gnomAD_E_AFR: 0.0457%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_EAS: 0.4874%</dd>
                                    <dd>gnomAD_E_FIN: 0.0224%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.1094%</dd>
                                    <dd>gnomAD_E_SAS: 0.0487%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_EAS: 0.7417%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PKN2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000065243">PKN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.513</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.327</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.513 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PKN2">PKN2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0005567, decreased circulating total protein level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.176 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PKN2">PKN2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKN2%0DNCKAP1L&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCKAP1L</a> and phenotypic similarity 0.474 to <a href="http://www.omim.org/entry/618982">Immunodeficiency 72 with autoinflammation</a> associated with NCKAP1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001903, Anemia - HP:0032184, Increased proportion of memory T cells</dd><dd>HP:0001541, Ascites - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0032184, Increased proportion of memory T cells</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0032184, Increased proportion of memory T cells</dd><dd>HP:0001396, Cholestasis - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001410, Decreased liver function - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0032184, Increased proportion of memory T cells</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0032184, Increased proportion of memory T cells</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKN2%0DNCKAP1L&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCKAP1L</a> and phenotypic similarity 0.712 to mouse mutant of NCKAP1L.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000693, spleen hyperplasia</dd><dd>HP:0001903, Anemia - MP:0002591, decreased mean corpuscular volume</dd><dd>HP:0001541, Ascites - MP:0000693, spleen hyperplasia</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008388, hypochromic microcytic anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0008801, abnormal erythroid progenitor cell morphology</dd><dd>HP:0001396, Cholestasis - MP:0000598, abnormal liver morphology</dd><dd>HP:0001410, Decreased liver function - MP:0001860, liver inflammation</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0001860, liver inflammation</dd><dd>HP:0001399, Hepatic failure - MP:0001860, liver inflammation</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0001853, heart inflammation</dd><dd>HP:0200114, Metabolic alkalosis - MP:0008596, increased circulating interleukin-6 level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0008596, increased circulating interleukin-6 level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0008596, increased circulating interleukin-6 level</dd><dd>HP:0001873, Thrombocytopenia - MP:0000208, decreased hematocrit</dd><dd>HP:0002151, Increased serum lactate - MP:0008596, increased circulating interleukin-6 level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0001860, liver inflammation</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0001860, liver inflammation</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000598, abnormal liver morphology</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.513</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.327</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-88770437-C-A [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs35207128">rs35207128</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.327</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370521.8">PKN2:ENST00000370521.8:c.590C&gt;A:p.(Ala197Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.71012676</dd>
                                    
                                    <dd>REVEL: 0.453</dd>
                                    <dd>MVP: 0.710</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2342%</dd>
                                    <dd>TOPMed: 0.0880%</dd>
                                    <dd>ExAC EAS: 1.2636%</dd>
                                    <dd>ExAC SAS: 0.0424%</dd>
                                    <dd>gnomAD_E_EAS: 1.2699%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0422%</dd>
                                    <dd>gnomAD_G_EAS: 1.6069%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="USP7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187555">USP7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.269</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.579</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.355 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=500055">Hao-Fountain syndrome due to 16p13.2 microdeletion</a> associated with USP7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0000135, Hypogonadism</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0000135, Hypogonadism</dd><dd>HP:0001410, Decreased liver function - HP:0000135, Hypogonadism</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0000135, Hypogonadism</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000135, Hypogonadism</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Phenotypic similarity 0.169 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=USP7">USP7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - ZP:0000038, pericardium edematous, abnormal</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.538 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP7%0DFBXO38&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBXO38</a> and phenotypic similarity 0.130 to <a href="http://www.omim.org/entry/615575">Neuronopathy, distal hereditary motor, type IID</a> associated with FBXO38.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0003431, Decreased motor nerve conduction velocity</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.538 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP7%0DFBXO38&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBXO38</a> and phenotypic similarity 0.603 to mouse mutant of FBXO38.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000601, small liver</dd><dd>HP:0001903, Anemia - MP:0010067, increased red blood cell distribution width</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0010067, increased red blood cell distribution width</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0002606, increased basophil cell number</dd><dd>HP:0001396, Cholestasis - MP:0000601, small liver</dd><dd>HP:0001410, Decreased liver function - MP:0000601, small liver</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000601, small liver</dd><dd>HP:0001399, Hepatic failure - MP:0000601, small liver</dd><dd>HP:0000938, Osteopenia - MP:0000063, decreased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation - MP:0002626, increased heart rate</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005343, increased circulating aspartate transaminase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002941, increased circulating alanine transaminase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002941, increased circulating alanine transaminase level</dd><dd>HP:0001873, Thrombocytopenia - MP:0002599, increased mean platelet volume</dd><dd>HP:0002151, Increased serum lactate - MP:0005343, increased circulating aspartate transaminase level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000601, small liver</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000601, small liver</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000601, small liver</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0005013, increased lymphocyte cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/616863">OMIM:616863</a> Hao-Fountain syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=500055">ORPHA:500055</a> Hao-Fountain syndrome due to 16p13.2 microdeletion (CNV)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.006</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.269</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.579</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>DEL</b>
                            <b>16-8894647-GA-G [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs373482138">rs373482138</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2098649%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.579</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344836.9">USP7:ENST00000344836.9:c.3112-8del:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3435%</dd>
                                    <dd>TOPMed: 0.3435%</dd>
                                    <dd>ExAC AFR: 0.0300%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 0.8429%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0485%</dd>
                                    <dd>gnomAD_E_AFR: 0.0395%</dd>
                                    <dd>gnomAD_E_AMR: 0.0061%</dd>
                                    <dd>gnomAD_E_EAS: 0.8011%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0185%</dd>
                                    <dd>gnomAD_E_SAS: 0.0394%</dd>
                                    <dd>gnomAD_G_AFR: 0.0577%</dd>
                                    <dd>gnomAD_G_EAS: 1.1125%</dd>
                                    <dd>gnomAD_G_OTH: 0.1035%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAH1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000114841">DNAH1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.327</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.497</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.327 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001746, Asplenia</dd><dd>HP:0001903, Anemia - HP:0001746, Asplenia</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001746, Asplenia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001746, Asplenia</dd><dd>HP:0001396, Cholestasis - HP:0001746, Asplenia</dd><dd>HP:0001410, Decreased liver function - HP:0001746, Asplenia</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001746, Asplenia</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0001746, Asplenia</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001746, Asplenia</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001669, Transposition of the great arteries</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.09 (p-value: 0.958770)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/617576">OMIM:617576</a> Spermatogenic failure 18 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/617577">OMIM:617577</a> Ciliary dyskinesia, primary, 37 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.327</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.497</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.327 to ORPHA:244 Primary ciliary dyskinesia</dt>
                                
                                    <dt>Phenotypic similarity 0.253 to OMIM:617577 Ciliary dyskinesia, primary, 37</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-52392945-A-G [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs375477856">rs375477856</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=965669%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.530</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420323.7">DNAH1:ENST00000420323.7:c.10394A&gt;G:p.(Asn3465Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.53119737</dd>
                                    
                                    <dd>REVEL: 0.272</dd>
                                    <dd>MVP: 0.531</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ESP EA: 0.0119%</dd>
                                    <dd>ESP All: 0.0079%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0134%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-52399696-T-C [0/1:0/1:0/0]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.464</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420323.7">DNAH1:ENST00000420323.7:c.12593T&gt;C:p.(Ile4198Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.46435556</dd>
                                    
                                    <dd>REVEL: 0.339</dd>
                                    <dd>MVP: 0.464</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FAM47A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185448">FAM47A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=6.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=6.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>X-34130760-CCGAGCGGAGACTGGACGTCCGACGAGTCTTGGGAGGCTG-C [1/1:0/0:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting, PM4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346193.5">FAM47A:ENST00000346193.5:c.1480_1518del:p.(Gln494_Ser506del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPANXB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000227234">SPANXB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=6.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=6.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>X-141002745-TCCAACGAGG-T [0/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1259923760">rs1259923760</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449283.2">SPANXB1:ENST00000449283.2:c.66_74del:p.(Glu26_Asn28del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=6.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>X-141002745-TCCAACGAGG-T [0/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1259923760">rs1259923760</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449283.2">SPANXB1:ENST00000449283.2:c.66_74del:p.(Glu26_Asn28del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ARHGAP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000089820">ARHGAP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=6.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.275</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARHGAP4%0DSLC12A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC12A1</a> and phenotypic similarity 0.629 to <a href="http://www.omim.org/entry/601678">Bartter syndrome, type 1</a> associated with SLC12A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0000843, Hyperparathyroidism</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0001944, Dehydration</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0000938, Osteopenia</dd><dd>HP:0001396, Cholestasis - HP:0000841, Hyperactive renin-angiotensin system</dd><dd>HP:0001410, Decreased liver function - HP:0000843, Hyperparathyroidism</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0000843, Hyperparathyroidism</dd><dd>HP:0000938, Osteopenia - HP:0000938, Osteopenia</dd><dd>HP:0001653, Mitral regurgitation - HP:0002632, Low-to-normal blood pressure</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001960, Hypokalemic metabolic alkalosis</dd><dd>HP:0003073, Hypoalbuminemia - HP:0000848, Increased circulating renin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003566, Increased serum prostaglandin E2</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0000848, Increased circulating renin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000848, Increased circulating renin level</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARHGAP4%0DSLC12A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC12A1</a> and phenotypic similarity 0.549 to mouse mutant of SLC12A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia - MP:0002608, increased hematocrit</dd><dd>HP:0001541, Ascites - MP:0001784, abnormal fluid regulation</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002608, increased hematocrit</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function - MP:0005582, increased renin activity</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0005582, increased renin activity</dd><dd>HP:0001399, Hepatic failure - MP:0005582, increased renin activity</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0012551, metabolic acidosis</dd><dd>HP:0003073, Hypoalbuminemia - MP:0005568, increased circulating total protein level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0005568, increased circulating total protein level</dd><dd>HP:0001873, Thrombocytopenia - MP:0002608, increased hematocrit</dd><dd>HP:0002151, Increased serum lactate - MP:0012551, metabolic acidosis</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0005582, increased renin activity</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0005582, increased renin activity</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0005560, decreased circulating glucose level</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0002608, increased hematocrit</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=6.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.275</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-153920694-C-T [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147143769">rs147143769</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.275</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000350060.10">ARHGAP4:ENST00000350060.10:c.613G&gt;A:p.(Ala205Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.27715346</dd>
                                    
                                    <dd>REVEL: 0.025</dd>
                                    <dd>MVP: 0.277</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0068%</dd>
                                    <dd>ESP EA: 0.0446%</dd>
                                    <dd>ESP All: 0.0284%</dd>
                                    <dd>ExAC NFE: 0.0065%</dd>
                                    <dd>gnomAD_E_NFE: 0.0064%</dd>
                                    <dd>gnomAD_G_AFR: 0.0167%</dd>
                                    <dd>gnomAD_G_NFE: 0.0096%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>X-153921709-G-A [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs377321582">rs377321582</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.088</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000350060.10">ARHGAP4:ENST00000350060.10:c.168C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.0833%</dd>
                            <dd>TOPMed: 0.0249%</dd>
                            <dd>UK10K: 0.0152%</dd>
                            <dd>ESP AA: 0.0262%</dd>
                            <dd>ESP All: 0.0095%</dd>
                            <dd>ExAC AFR: 0.1472%</dd>
                            <dd>ExAC EAS: 0.2674%</dd>
                            <dd>ExAC NFE: 0.0162%</dd>
                            <dd>ExAC OTH: 0.6494%</dd>
                            <dd>ExAC SAS: 0.0146%</dd>
                            <dd>gnomAD_E_AFR: 0.0790%</dd>
                            <dd>gnomAD_E_AMR: 0.0119%</dd>
                            <dd>gnomAD_E_EAS: 0.2041%</dd>
                            <dd>gnomAD_E_NFE: 0.0077%</dd>
                            <dd>gnomAD_E_OTH: 0.2518%</dd>
                            <dd>gnomAD_E_SAS: 0.0231%</dd>
                            <dd>gnomAD_G_AFR: 0.0502%</dd>
                            <dd>gnomAD_G_EAS: 0.1921%</dd>
                            <dd>gnomAD_G_OTH: 0.1391%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PAPPA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000116183">PAPPA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=6.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.251</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.502 to <a href="http://www.omim.org/entry/619489">Short stature, Dauber-Argente type</a> associated with PAPPA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0004349, Reduced bone mineral density</dd><dd>HP:0001396, Cholestasis - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0001410, Decreased liver function - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0000938, Osteopenia - HP:0000938, Osteopenia</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0034184, Increased insulin like growth factor binding protein acid labile subunit concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0034184, Increased insulin like growth factor binding protein acid labile subunit concentration</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0008283, Fasting hyperinsulinemia</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.488 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PAPPA2">PAPPA2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production - MP:0004952, increased spleen weight</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0002981, increased liver weight</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002981, increased liver weight</dd><dd>HP:0001399, Hepatic failure - MP:0002981, increased liver weight</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0002176, increased brain weight</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0002981, increased liver weight</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PAPPA2%0DCTSG&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CTSG</a> and phenotypic similarity 0.622 to mouse mutant of CTSG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia - MP:0001577, anemia</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production - MP:0001601, abnormal myelopoiesis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0000218, increased leukocyte cell number</dd><dd>HP:0001396, Cholestasis - MP:0000599, enlarged liver</dd><dd>HP:0001410, Decreased liver function - MP:0000599, enlarged liver</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000599, enlarged liver</dd><dd>HP:0001399, Hepatic failure - MP:0000599, enlarged liver</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000599, enlarged liver</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000599, enlarged liver</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000599, enlarged liver</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000218, increased leukocyte cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619489">OMIM:619489</a> Short stature, Dauber-Argente type - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=6.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.251</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.502 to OMIM:619489 Short stature, Dauber-Argente type</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-176594926-G-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs374840800">rs374840800</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2194332%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.402</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367662.5">PAPPA2:ENST00000367662.5:c.1322G&gt;A:p.(Ser441Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4081723</dd>
                                    
                                    <dd>REVEL: 0.100</dd>
                                    <dd>MVP: 0.408</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC EAS: 0.0348%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_EAS: 0.0290%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-176702685-T-G [0/1:0/0:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367662.5">PAPPA2:ENST00000367662.5:c.3315T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT84" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161849">KRT84</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=6.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=6.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.798</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-52381507-A-G [1/1:0/1:0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000257951.3">KRT84:ENST00000257951.3:c.931T&gt;C:p.(Ser311Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.79798794</dd>
                                    
                                    <dd>REVEL: 0.455</dd>
                                    <dd>MVP: 0.798</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GOLGA6A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000159289">GOLGA6A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=6.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=6.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.797</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>15-74071548-G-A [0/1:./.:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1219693041">rs1219693041</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000290438.3">GOLGA6A:ENST00000290438.3:c.1696+6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0156%</dd>
                                    <dd>TOPMed: 0.0156%</dd>
                                    <dd>gnomAD_E_SAS: 0.0157%</dd>
                                    <dd>gnomAD_G_AFR: 0.0241%</dd>
                                    <dd>gnomAD_G_NFE: 0.0073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRAMEF15" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204501">PRAMEF15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.386</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.346</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.386 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PRAMEF15">PRAMEF15</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000691, enlarged spleen</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0000691, enlarged spleen</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000689, abnormal spleen morphology</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0001410, Decreased liver function - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0001399, Hepatic failure - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0001944, abnormal pancreas morphology</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=6.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.386</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.346</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-13322125-T-G [0/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1272016578">rs1272016578</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.346</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376152.2">PRAMEF15:ENST00000376152.2:c.1298T&gt;G:p.(Phe433Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.34563237</dd>
                                    
                                    <dd>REVEL: 0.035</dd>
                                    <dd>MVP: 0.346</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>1-13322094-G-A [0/1:0/0:./.]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1200491508">rs1200491508</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.040</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376152.2">PRAMEF15:ENST00000376152.2:c.1267G&gt;A:p.(Gly423Ser)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            <dd>Best Score: 0.04010828</dd>
                            
                            <dd>REVEL: 0.010</dd>
                            <dd>MVP: 0.040</dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MRPL45" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000278845">MRPL45</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=6.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.207</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MRPL45%0DSMC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMC5</a> and phenotypic similarity 0.510 to <a href="http://www.omim.org/entry/620185">Atelis syndrome 2</a> associated with SMC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0000771, Gynecomastia</dd><dd>HP:0001903, Anemia - HP:0001903, Anemia</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001903, Anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001903, Anemia</dd><dd>HP:0001396, Cholestasis - HP:0002925, Elevated circulating thyroid-stimulating hormone concentration</dd><dd>HP:0001410, Decreased liver function - HP:0002925, Elevated circulating thyroid-stimulating hormone concentration</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0002925, Elevated circulating thyroid-stimulating hormone concentration</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0001642, Pulmonic stenosis</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000842, Hyperinsulinemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0000842, Hyperinsulinemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0000842, Hyperinsulinemia</dd><dd>HP:0001873, Thrombocytopenia - HP:0001873, Thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - HP:0000842, Hyperinsulinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000842, Hyperinsulinemia</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001903, Anemia</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MRPL45%0DSMC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMC5</a> and phenotypic similarity 0.601 to mouse mutant of SMC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0005011, increased eosinophil cell number</dd><dd>HP:0001903, Anemia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0005011, increased eosinophil cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0005016, decreased lymphocyte cell number</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0005560, decreased circulating glucose level</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=6.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.207</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-38297226-T-G [1/1:./.:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs9890011">rs9890011</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.207</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000613675.5">MRPL45:ENST00000613675.5:c.43T&gt;G:p.(Phe15Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25778696</dd>
                                    
                                    <dd>REVEL: 0.102</dd>
                                    <dd>MVP: 0.258</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4685%</dd>
                                    <dd>TOPMed: 0.2803%</dd>
                                    <dd>gnomAD_E_AFR: 0.8131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0687%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.1097%</dd>
                                    <dd>gnomAD_G_AFR: 0.8953%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EXOC6B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144036">EXOC6B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=6.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.207</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.234 to <a href="http://www.omim.org/entry/618395">Spondyloepimetaphyseal dysplasia with joint laxity, type 3</a> associated with EXOC6B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001216, Delayed ossification of carpal bones</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia - HP:0001216, Delayed ossification of carpal bones</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001513, Obesity</dd></dl><dl><dt>Phenotypic similarity 0.152 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EXOC6B">EXOC6B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0005560, decreased circulating glucose level</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EXOC6B%0DRABL2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RABL2A</a> and phenotypic similarity 0.620 to mouse mutant of RABL2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0002981, increased liver weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0002981, increased liver weight</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0010124, decreased bone mineral content</dd><dd>HP:0001396, Cholestasis - MP:0002981, increased liver weight</dd><dd>HP:0001410, Decreased liver function - MP:0002981, increased liver weight</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002981, increased liver weight</dd><dd>HP:0001399, Hepatic failure - MP:0002981, increased liver weight</dd><dd>HP:0000938, Osteopenia - MP:0000063, decreased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation - MP:0006203, eye hemorrhage</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002079, increased circulating insulin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002079, increased circulating insulin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0002079, increased circulating insulin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002981, increased liver weight</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0002981, increased liver weight</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618395">OMIM:618395</a> Spondyloepimetaphyseal dysplasia with joint laxity, type 3 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=6.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.207</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.234 to OMIM:618395 Spondyloepimetaphyseal dysplasia with joint laxity, type 3</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>2-72492312-G-C [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs181182127">rs181182127</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=730161%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.207</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000272427.11">EXOC6B:ENST00000272427.11:c.1665+6C&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3123%</dd>
                                    <dd>TOPMed: 0.3123%</dd>
                                    <dd>ExAC EAS: 1.8686%</dd>
                                    <dd>ExAC OTH: 0.1121%</dd>
                                    <dd>ExAC SAS: 0.0245%</dd>
                                    <dd>gnomAD_E_EAS: 1.6943%</dd>
                                    <dd>gnomAD_E_OTH: 0.0367%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                    <dd>gnomAD_G_EAS: 1.7348%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KCNN4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104783">KCNN4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=7.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.594</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.077</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.657 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3202">Dehydrated hereditary stomatocytosis</a> associated with KCNN4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0001878, Hemolytic anemia</dd><dd>HP:0001541, Ascites - HP:0000969, Edema</dd><dd>HP:0010972, Anemia of inadequate production - HP:0010972, Anemia of inadequate production</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0010972, Anemia of inadequate production</dd><dd>HP:0001396, Cholestasis - HP:0001046, Intermittent jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0001081, Cholelithiasis</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0025435, Increased circulating lactate dehydrogenase concentration</dd><dd>HP:0001399, Hepatic failure - HP:0001081, Cholelithiasis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0030950, Pulmonary venous hypertension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000969, Edema</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003281, Increased circulating ferritin concentration</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003281, Increased circulating ferritin concentration</dd><dd>HP:0001873, Thrombocytopenia - HP:0001901, Polycythemia</dd><dd>HP:0002151, Increased serum lactate - HP:0003573, Increased total bilirubin</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0025435, Increased circulating lactate dehydrogenase concentration</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0025435, Increased circulating lactate dehydrogenase concentration</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0030242, Portal vein thrombosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0010972, Anemia of inadequate production</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.513 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KCNN4">KCNN4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000691, enlarged spleen</dd><dd>HP:0001903, Anemia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0001541, Ascites - MP:0000691, enlarged spleen</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002874, decreased hemoglobin content</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0000598, abnormal liver morphology</dd><dd>HP:0001410, Decreased liver function - MP:0000598, abnormal liver morphology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000598, abnormal liver morphology</dd><dd>HP:0001399, Hepatic failure - MP:0000598, abnormal liver morphology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002874, decreased hemoglobin content</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002874, decreased hemoglobin content</dd><dd>HP:0001873, Thrombocytopenia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000601, small liver</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0010067, increased red blood cell distribution width</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNN4%0DIKZF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IKZF1</a> and phenotypic similarity 0.710 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=36426">Stevens-Johnson syndrome</a> associated with IKZF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0006554, Acute hepatic failure</dd><dd>HP:0001903, Anemia - HP:0001903, Anemia</dd><dd>HP:0001541, Ascites - HP:0003781, Excessive salivation</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001903, Anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001874, Abnormality of neutrophils</dd><dd>HP:0001396, Cholestasis - HP:0006554, Acute hepatic failure</dd><dd>HP:0001410, Decreased liver function - HP:0006554, Acute hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0006554, Acute hepatic failure</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0001658, Myocardial infarction</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001960, Hypokalemic metabolic alkalosis</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0001873, Thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - HP:0001960, Hypokalemic metabolic alkalosis</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0006554, Acute hepatic failure</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001874, Abnormality of neutrophils</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001824, Weight loss</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNN4%0DIKZF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IKZF1</a> and phenotypic similarity 0.592 to mouse mutant of IKZF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000600, liver hypoplasia</dd><dd>HP:0001903, Anemia - MP:0001577, anemia</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0001601, abnormal myelopoiesis</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0011176, abnormal erythroblast morphology</dd><dd>HP:0001396, Cholestasis - MP:0000600, liver hypoplasia</dd><dd>HP:0001410, Decreased liver function - MP:0000600, liver hypoplasia</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000600, liver hypoplasia</dd><dd>HP:0001399, Hepatic failure - MP:0000600, liver hypoplasia</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002874, decreased hemoglobin content</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002874, decreased hemoglobin content</dd><dd>HP:0001873, Thrombocytopenia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000600, liver hypoplasia</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000600, liver hypoplasia</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000600, liver hypoplasia</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0005425, increased macrophage cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNN4%0DIKZF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IKZF1</a> and phenotypic similarity 0.336 to fish mutant of IKZF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0001903, Anemia - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - ZP:0010760, lymphoid lineage cell migration into thymus involved in thymus epithelium morphogenesis disrupted, abnormal</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0001396, Cholestasis - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0001410, Decreased liver function - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0002910, Elevated hepatic transaminase - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0001399, Hepatic failure - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - ZP:0010760, lymphoid lineage cell migration into thymus involved in thymus epithelium morphogenesis disrupted, abnormal</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0004774, thymus lacks parts or has fewer parts of type lymphocyte, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 1.20 (p-value: 0.000010)</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616689">OMIM:616689</a> Dehydrated hereditary stomatocytosis 2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3202">ORPHA:3202</a> Dehydrated hereditary stomatocytosis - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=586">ORPHA:586</a> Cystic fibrosis (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=7.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.594</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.077</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.594 to ORPHA:586 Cystic fibrosis</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-43780712-C-T [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200169896">rs200169896</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.081</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648319.1">KCNN4:ENST00000648319.1:c.150G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0781%</dd>
                                    <dd>TOPMed: 0.0781%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC EAS: 0.5666%</dd>
                                    <dd>ExAC OTH: 0.2203%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_EAS: 0.6320%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0731%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.8642%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1025%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-43774371-G-A [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs551228061">rs551228061</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=728249%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.072</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000648319.1">KCNN4:ENST00000648319.1:c.504C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0468%</dd>
                                    <dd>TOPMed: 0.0366%</dd>
                                    <dd>ExAC AFR: 0.0169%</dd>
                                    <dd>ExAC EAS: 0.7015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0153%</dd>
                                    <dd>gnomAD_E_EAS: 0.6373%</dd>
                                    <dd>gnomAD_G_EAS: 1.1111%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MUC3A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169894">MUC3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.725</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.725</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>7-100957124-TAGG-T [0/1:./.:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1422404475">rs1422404475</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.5346_5348del:p.(Gly1783del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-100953812-T-C [0/1:./.:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1385924525">rs1385924525</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.600</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.2033T&gt;C:p.(Val678Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.600</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-100953755-C-G [0/1:0/0:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs879161245">rs879161245</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.600</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1976C&gt;G:p.(Ser659Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953697-T-C [0/1:./.:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1298273392">rs1298273392</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1918T&gt;C:p.(Ser640Pro)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953728-C-T [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs982221309">rs982221309</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1949C&gt;T:p.(Thr650Met)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953770-C-G [0/1:0/0:./.]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1289165986">rs1289165986</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1991C&gt;G:p.(Thr664Ser)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953776-T-A [0/1:0/0:./.]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1307985701">rs1307985701</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1997T&gt;A:p.(Met666Lys)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953779-C-T [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1375512213">rs1375512213</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.2000C&gt;T:p.(Pro667Leu)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955779-C-A [1/1:./.:./.]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1313584776">rs1313584776</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4000C&gt;A:p.(Pro1334Thr)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955888-C-T [0/1:0/1:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1443550411">rs1443550411</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4109C&gt;T:p.(Thr1370Ile)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955950-A-G [0/1:0/1:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1380133592">rs1380133592</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4171A&gt;G:p.(Ile1391Val)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100956008-C-G [0/1:0/1:./.]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1314196169">rs1314196169</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4229C&gt;G:p.(Thr1410Ser)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">MISSENSE_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100956269-C-T [0/1:0/0:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1435832477">rs1435832477</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.600</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4490C&gt;T:p.(Pro1497Leu)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953741-C-T [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1181790485">rs1181790485</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.1962C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100953789-T-C [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1457246080">rs1457246080</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.2010T&gt;C:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100954830-C-G [0/1:./.:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1310488669">rs1310488669</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.3051C&gt;G:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955826-T-G [0/1:0/1:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1394171058">rs1394171058</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4047T&gt;G:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955898-T-C [0/1:0/1:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1346410915">rs1346410915</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4119T&gt;C:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955934-C-T [0/1:0/1:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1372154399">rs1372154399</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4155C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100955991-C-T [0/1:0/1:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs1213586128">rs1213586128</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.4212C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100965843-G-A [0/1:0/0:0/0]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs67377634">rs67377634</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9588G&gt;A:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            <dd>No frequency data</dd>
                            
                        </dl>
                    </div>
                </div>
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>7-100965798-G-T [0/1:0/0:0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs35790618">rs35790618</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.100</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379458.9">MUC3A:ENST00000379458.9:c.9543G&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>1000Genomes: 0.0200%</dd>
                            <dd>TOPMed: 0.0200%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CSRNP2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110925">CSRNP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.128</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSRNP2%0DPHKB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PHKB</a> and phenotypic similarity 0.685 to <a href="http://www.omim.org/entry/261750">Phosphorylase kinase deficiency of liver and muscle, autosomal recessive</a> associated with PHKB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0001744, Splenomegaly</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001744, Splenomegaly</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001744, Splenomegaly</dd><dd>HP:0001396, Cholestasis - HP:0002240, Hepatomegaly</dd><dd>HP:0001410, Decreased liver function - HP:0002240, Hepatomegaly</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0002240, Hepatomegaly</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001943, Hypoglycemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0002149, Hyperuricemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0002149, Hyperuricemia</dd><dd>HP:0001873, Thrombocytopenia - HP:0001744, Splenomegaly</dd><dd>HP:0002151, Increased serum lactate - HP:0001943, Hypoglycemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0006568, Increased hepatic glycogen content</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001744, Splenomegaly</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CSRNP2%0DPHKB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PHKB</a> and phenotypic similarity 0.661 to mouse mutant of PHKB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0000598, abnormal liver morphology</dd><dd>HP:0001410, Decreased liver function - MP:0000598, abnormal liver morphology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000598, abnormal liver morphology</dd><dd>HP:0001399, Hepatic failure - MP:0000598, abnormal liver morphology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0001552, increased circulating triglyceride level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000598, abnormal liver morphology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0010400, increased liver glycogen level</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.128</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>12-51064672-G-A [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148790966">rs148790966</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.128</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228515.6">CSRNP2:ENST00000228515.6:c.709-3C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3592%</dd>
                                    <dd>TOPMed: 0.0748%</dd>
                                    <dd>ExAC EAS: 1.7751%</dd>
                                    <dd>ExAC SAS: 0.0567%</dd>
                                    <dd>gnomAD_E_AMR: 0.0047%</dd>
                                    <dd>gnomAD_E_EAS: 1.8941%</dd>
                                    <dd>gnomAD_E_OTH: 0.1710%</dd>
                                    <dd>gnomAD_E_SAS: 0.0506%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_EAS: 1.9753%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLXNA3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130827">PLXNA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.122</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.241 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PLXNA3">PLXNA3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008960, abnormal axon pruning</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0008960, abnormal axon pruning</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.330 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PLXNA3">PLXNA3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - ZP:0012648, axon spinal cord increased branchiness, abnormal</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNA3%0DSEMA4D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEMA4D</a> and phenotypic similarity 0.727 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171">Primary sclerosing cholangitis</a> associated with SEMA4D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001433, Hepatosplenomegaly</dd><dd>HP:0001903, Anemia - HP:0001879, Abnormal eosinophil morphology</dd><dd>HP:0001541, Ascites - HP:0001541, Ascites</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001879, Abnormal eosinophil morphology</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001879, Abnormal eosinophil morphology</dd><dd>HP:0001396, Cholestasis - HP:0001396, Cholestasis</dd><dd>HP:0001410, Decreased liver function - HP:0100626, Chronic hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0100626, Chronic hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0000938, Osteopenia</dd><dd>HP:0001653, Mitral regurgitation - HP:0001635, Congestive heart failure</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001945, Fever</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003073, Hypoalbuminemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0100512, Low levels of vitamin D</dd><dd>HP:0001873, Thrombocytopenia - HP:0001879, Abnormal eosinophil morphology</dd><dd>HP:0002151, Increased serum lactate - HP:0003073, Hypoalbuminemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001394, Cirrhosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0100727, Histiocytosis</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001824, Weight loss</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNA3%0DSEMA4D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEMA4D</a> and phenotypic similarity 0.438 to mouse mutant of SEMA4D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000691, enlarged spleen</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0000691, enlarged spleen</dd><dd>HP:0010972, Anemia of inadequate production - MP:0000691, enlarged spleen</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0010124, decreased bone mineral content</dd><dd>HP:0001396, Cholestasis - MP:0000691, enlarged spleen</dd><dd>HP:0001410, Decreased liver function - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0001399, Hepatic failure - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0000938, Osteopenia - MP:0010124, decreased bone mineral content</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0002968, increased circulating alkaline phosphatase level</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000691, enlarged spleen</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.122</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-154464030-C-T [1/1:./.:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369262880">rs369262880</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2440101%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.122</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369682.4">PLXNA3:ENST00000369682.4:c.1627C&gt;T:p.(Arg543Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.123314135</dd>
                                    
                                    <dd>REVEL: 0.083</dd>
                                    <dd>MVP: 0.123</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0208%</dd>
                                    <dd>TOPMed: 0.0042%</dd>
                                    <dd>UK10K: 0.0152%</dd>
                                    <dd>ExAC AFR: 0.0315%</dd>
                                    <dd>ExAC EAS: 0.1020%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_AFR: 0.0085%</dd>
                                    <dd>gnomAD_E_EAS: 0.0560%</dd>
                                    <dd>gnomAD_E_OTH: 0.0256%</dd>
                                    <dd>gnomAD_E_SAS: 0.0110%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF205" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000122386">ZNF205</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.096</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.376 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZNF205">ZNF205</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia - MP:0002874, decreased hemoglobin content</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0002874, decreased hemoglobin content</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005554, decreased circulating creatinine level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002874, decreased hemoglobin content</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002874, decreased hemoglobin content</dd><dd>HP:0001873, Thrombocytopenia - MP:0002874, decreased hemoglobin content</dd><dd>HP:0002151, Increased serum lactate - MP:0005554, decreased circulating creatinine level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0002874, decreased hemoglobin content</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF205%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.592 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=342">Familial Mediterranean fever</a> associated with MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0002716, Lymphadenopathy</dd><dd>HP:0001541, Ascites - HP:0001541, Ascites</dd><dd>HP:0010972, Anemia of inadequate production - HP:0002716, Lymphadenopathy</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0002716, Lymphadenopathy</dd><dd>HP:0001396, Cholestasis - HP:0006554, Acute hepatic failure</dd><dd>HP:0001410, Decreased liver function - HP:0006554, Acute hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0006554, Acute hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0001369, Arthritis</dd><dd>HP:0001653, Mitral regurgitation - HP:0001658, Myocardial infarction</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001945, Fever</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0002716, Lymphadenopathy</dd><dd>HP:0002151, Increased serum lactate - HP:0001945, Fever</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0006554, Acute hepatic failure</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0002633, Vasculitis</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF205%0DMEFV&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MEFV</a> and phenotypic similarity 0.615 to mouse mutant of MEFV.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000691, enlarged spleen</dd><dd>HP:0001903, Anemia - MP:0001577, anemia</dd><dd>HP:0001541, Ascites - MP:0000691, enlarged spleen</dd><dd>HP:0010972, Anemia of inadequate production - MP:0001577, anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0000218, increased leukocyte cell number</dd><dd>HP:0001396, Cholestasis - MP:0001860, liver inflammation</dd><dd>HP:0001410, Decreased liver function - MP:0001860, liver inflammation</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0001860, liver inflammation</dd><dd>HP:0001399, Hepatic failure - MP:0001860, liver inflammation</dd><dd>HP:0000938, Osteopenia - MP:0002933, joint inflammation</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0001873, Thrombocytopenia - MP:0000322, increased granulocyte number</dd><dd>HP:0002151, Increased serum lactate - MP:0008553, increased circulating tumor necrosis factor level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0001860, liver inflammation</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0001860, liver inflammation</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0001860, liver inflammation</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0005425, increased macrophage cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001732, postnatal growth retardation</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.096</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-3119854-C-T [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs376806635">rs376806635</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000219091.9">ZNF205:ENST00000219091.9:c.1194C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0000%</dd>
                                    <dd>TOPMed: 0.0083%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 0.1754%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.1747%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0130%</dd>
                                    <dd>gnomAD_G_EAS: 0.2469%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLXNB3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198753">PLXNB3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNB3%0DABCD3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABCD3</a> and phenotypic similarity 0.681 to <a href="http://www.omim.org/entry/616278">?Bile acid synthesis defect, congenital, 5</a> associated with ABCD3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0001891, Iron deficiency anemia</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001891, Iron deficiency anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001891, Iron deficiency anemia</dd><dd>HP:0001396, Cholestasis - HP:0000952, Jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0001409, Portal hypertension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0002904, Hyperbilirubinemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0002904, Hyperbilirubinemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0012202, Increased serum bile acid concentration</dd><dd>HP:0001873, Thrombocytopenia - HP:0001891, Iron deficiency anemia</dd><dd>HP:0002151, Increased serum lactate - HP:0002904, Hyperbilirubinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001395, Hepatic fibrosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001891, Iron deficiency anemia</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PLXNB3%0DABCD3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABCD3</a> and phenotypic similarity 0.513 to mouse mutant of ABCD3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0004773, abnormal bile composition</dd><dd>HP:0001410, Decreased liver function - MP:0004773, abnormal bile composition</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0004773, abnormal bile composition</dd><dd>HP:0001399, Hepatic failure - MP:0004773, abnormal bile composition</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0004773, abnormal bile composition</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0004773, abnormal bile composition</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000599, enlarged liver</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>X-153770151-G-A [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201004092">rs201004092</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361971.10">PLXNB3:ENST00000361971.10:c.1689G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0624%</dd>
                                    <dd>TOPMed: 0.0087%</dd>
                                    <dd>UK10K: 0.0152%</dd>
                                    <dd>ExAC EAS: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0148%</dd>
                                    <dd>gnomAD_E_EAS: 0.0078%</dd>
                                    <dd>gnomAD_E_FIN: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0150%</dd>
                                    <dd>gnomAD_G_NFE: 0.0855%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AGAP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188234">AGAP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.104</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AGAP4%0DSEC23B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEC23B</a> and phenotypic similarity 0.604 to <a href="http://www.omim.org/entry/224100">Dyserythropoietic anemia, congenital, type II</a> associated with SEC23B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0001923, Reticulocytosis</dd><dd>HP:0001541, Ascites - HP:0001744, Splenomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0010972, Anemia of inadequate production</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0010972, Anemia of inadequate production</dd><dd>HP:0001396, Cholestasis - HP:0000952, Jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0001081, Cholelithiasis</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0003655, Reduced level of N-acetylglucosaminyltransferase II</dd><dd>HP:0001399, Hepatic failure - HP:0001081, Cholelithiasis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - HP:0003655, Reduced level of N-acetylglucosaminyltransferase II</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0003655, Reduced level of N-acetylglucosaminyltransferase II</dd><dd>HP:0001873, Thrombocytopenia - HP:0001923, Reticulocytosis</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0003655, Reduced level of N-acetylglucosaminyltransferase II</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0003655, Reduced level of N-acetylglucosaminyltransferase II</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001081, Cholelithiasis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0010972, Anemia of inadequate production</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AGAP4%0DSEC23B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEC23B</a> and phenotypic similarity 0.351 to mouse mutant of SEC23B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0009165, abnormal endocrine pancreas morphology</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0010148, abnormal exocrine pancreas physiology</dd><dd>HP:0001410, Decreased liver function - MP:0010148, abnormal exocrine pancreas physiology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0010148, abnormal exocrine pancreas physiology</dd><dd>HP:0001399, Hepatic failure - MP:0010148, abnormal exocrine pancreas physiology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0010148, abnormal exocrine pancreas physiology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0010148, abnormal exocrine pancreas physiology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0009177, decreased pancreatic alpha cell number</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.104</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-45826783-T-C [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1282313249">rs1282313249</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.112</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616763.6">AGAP4:ENST00000616763.6:c.1193A&gt;G:p.(Asn398Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11264884</dd>
                                    
                                    <dd>REVEL: 0.090</dd>
                                    <dd>MVP: 0.113</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_EAS: 0.0530%</dd>
                                    <dd>gnomAD_E_SAS: 0.0579%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-45826199-G-A [0/1:./.:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1483564807">rs1483564807</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.097</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616763.6">AGAP4:ENST00000616763.6:c.1777C&gt;T:p.(Arg593Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.097</dd>
                                    
                                    <dd>REVEL: 0.097</dd>
                                    <dd>MVP: 0.095</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.097</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-45826199-G-A [0/1:./.:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1483564807">rs1483564807</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.097</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000616763.6">AGAP4:ENST00000616763.6:c.1777C&gt;T:p.(Arg593Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.097</dd>
                                    
                                    <dd>REVEL: 0.097</dd>
                                    <dd>MVP: 0.095</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="VCX3B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000205642">VCX3B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VCX3B%0DIRS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRS1</a> and phenotypic similarity 0.258 to <a href="http://www.omim.org/entry/125853">Type 2 diabetes mellitus, susceptibility to</a> associated with IRS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0001410, Decreased liver function - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000855, Insulin resistance</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0000855, Insulin resistance</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0005978, Type II diabetes mellitus</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0031819, Increased waist to hip ratio</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VCX3B%0DIRS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRS1</a> and phenotypic similarity 0.638 to mouse mutant of IRS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0001903, Anemia - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0004989, decreased osteoblast cell number</dd><dd>HP:0001396, Cholestasis - MP:0003059, decreased insulin secretion</dd><dd>HP:0001410, Decreased liver function - MP:0003059, decreased insulin secretion</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0003059, decreased insulin secretion</dd><dd>HP:0001399, Hepatic failure - MP:0003059, decreased insulin secretion</dd><dd>HP:0000938, Osteopenia - MP:0000063, decreased bone mineral density</dd><dd>HP:0001653, Mitral regurgitation - MP:0006143, increased systemic arterial diastolic blood pressure</dd><dd>HP:0200114, Metabolic alkalosis - MP:0002079, increased circulating insulin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002079, increased circulating insulin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0002151, Increased serum lactate - MP:0001554, increased circulating free fatty acids level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0003059, decreased insulin secretion</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0003059, decreased insulin secretion</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0003059, decreased insulin secretion</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0008396, abnormal osteoclast differentiation</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VCX3B%0DIRS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IRS1</a> and phenotypic similarity 0.299 to fish mutant of IRS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - ZP:0005798, developmental growth decreased rate, abnormal</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>X-8466206-C-T [0/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201851798">rs201851798</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381032.6">VCX3B:ENST00000381032.6:c.564C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0130%</dd>
                                    <dd>gnomAD_E_AMR: 0.0092%</dd>
                                    <dd>gnomAD_E_EAS: 0.0513%</dd>
                                    <dd>gnomAD_E_NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_SAS: 0.0069%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>X-8466206-C-T [0/1:0/0:./.]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201851798">rs201851798</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381032.6">VCX3B:ENST00000381032.6:c.564C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0130%</dd>
                                    <dd>gnomAD_E_AMR: 0.0092%</dd>
                                    <dd>gnomAD_E_EAS: 0.0513%</dd>
                                    <dd>gnomAD_E_NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_SAS: 0.0069%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADAMTS5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154736">ADAMTS5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.098</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.175 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ADAMTS5">ADAMTS5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia - MP:0003436, decreased susceptibility to induced arthritis</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADAMTS5%0DIL18BP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL18BP</a> and phenotypic similarity 0.601 to <a href="http://www.omim.org/entry/618549">?Hepatitis, fulminant viral, susceptibility to</a> associated with IL18BP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0002240, Hepatomegaly</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0000952, Jaundice</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0000225, Gingival bleeding</dd><dd>HP:0200114, Metabolic alkalosis - HP:0100651, Type I diabetes mellitus</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0100651, Type I diabetes mellitus</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - HP:0002910, Elevated hepatic transaminase</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0002240, Hepatomegaly</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADAMTS5%0DIL18BP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IL18BP</a> and phenotypic similarity 0.561 to mouse mutant of IL18BP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia - MP:0005152, pancytopenia</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production - MP:0005152, pancytopenia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0005152, pancytopenia</dd><dd>HP:0001396, Cholestasis - MP:0000599, enlarged liver</dd><dd>HP:0001410, Decreased liver function - MP:0000599, enlarged liver</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000599, enlarged liver</dd><dd>HP:0001399, Hepatic failure - MP:0000599, enlarged liver</dd><dd>HP:0000938, Osteopenia - MP:0005152, pancytopenia</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0008577, increased circulating interferon-gamma level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0011890, increased circulating ferritin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0011890, increased circulating ferritin level</dd><dd>HP:0001873, Thrombocytopenia - MP:0005152, pancytopenia</dd><dd>HP:0002151, Increased serum lactate - MP:0008577, increased circulating interferon-gamma level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000599, enlarged liver</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000599, enlarged liver</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0000599, enlarged liver</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0005425, increased macrophage cell number</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001263, weight loss</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.098</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>21-26924251-A-G [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201648090">rs201648090</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.098</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284987.6">ADAMTS5:ENST00000284987.6:c.2595T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1093%</dd>
                                    <dd>TOPMed: 0.0042%</dd>
                                    <dd>ExAC EAS: 0.1387%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_EAS: 0.1160%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CACNA1E" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198216">CACNA1E</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=7.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.378</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.317 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CACNA1E">CACNA1E</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0009177, decreased pancreatic alpha cell number</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0009177, decreased pancreatic alpha cell number</dd><dd>HP:0001410, Decreased liver function - MP:0003562, abnormal pancreatic beta cell physiology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0003562, abnormal pancreatic beta cell physiology</dd><dd>HP:0001399, Hepatic failure - MP:0003562, abnormal pancreatic beta cell physiology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002696, decreased circulating glucagon level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002696, decreased circulating glucagon level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0003562, abnormal pancreatic beta cell physiology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0003562, abnormal pancreatic beta cell physiology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0003565, abnormal glucagon secretion</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA1E%0DCLCN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CLCN2</a> and phenotypic similarity 0.614 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=404">Familial hyperaldosteronism type II</a> associated with CLCN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0008221, Adrenal hyperplasia</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0008221, Adrenal hyperplasia</dd><dd>HP:0001410, Decreased liver function - HP:0011740, Glucocortocoid-insensitive primary hyperaldosteronism</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0011740, Glucocortocoid-insensitive primary hyperaldosteronism</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0000822, Hypertension</dd><dd>HP:0200114, Metabolic alkalosis - HP:0200114, Metabolic alkalosis</dd><dd>HP:0003073, Hypoalbuminemia - HP:0040084, Abnormal circulating renin</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0011740, Glucocortocoid-insensitive primary hyperaldosteronism</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0200114, Metabolic alkalosis</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0008221, Adrenal hyperplasia</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0002170, Intracranial hemorrhage</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA1E%0DCLCN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CLCN2</a> and phenotypic similarity 0.386 to mouse mutant of CLCN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0004109, abnormal Sertoli cell development</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0031384, impaired blood-brain barrier function</dd><dd>HP:0200114, Metabolic alkalosis - MP:0003352, increased circulating renin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0003352, increased circulating renin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0003352, increased circulating renin level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0003352, increased circulating renin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0000955, abnormal spinal cord morphology</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618285">OMIM:618285</a> Developmental and epileptic encephalopathy 69 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.378</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-181733741-G-A [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs768308084">rs768308084</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1448977%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.657</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367573.7">CACNA1E:ENST00000367573.7:c.3253G&gt;A:p.(Val1085Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6580741</dd>
                                    
                                    <dd>REVEL: 0.374</dd>
                                    <dd>MVP: 0.658</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC FIN: 0.0164%</dd>
                                    <dd>ExAC NFE: 0.0033%</dd>
                                    <dd>gnomAD_E_FIN: 0.0051%</dd>
                                    <dd>gnomAD_E_NFE: 0.0023%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-181762642-C-T [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs549521037">rs549521037</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2140204%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367573.7">CACNA1E:ENST00000367573.7:c.4674C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0156%</dd>
                                    <dd>TOPMed: 0.0004%</dd>
                                    <dd>ExAC EAS: 0.0236%</dd>
                                    <dd>gnomAD_E_EAS: 0.0178%</dd>
                                    <dd>gnomAD_E_OTH: 0.0187%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAGEC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155495">MAGEC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=8.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.090</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAGEC1%0DSMC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMC5</a> and phenotypic similarity 0.510 to <a href="http://www.omim.org/entry/620185">Atelis syndrome 2</a> associated with SMC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0000771, Gynecomastia</dd><dd>HP:0001903, Anemia - HP:0001903, Anemia</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001903, Anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0001903, Anemia</dd><dd>HP:0001396, Cholestasis - HP:0002925, Elevated circulating thyroid-stimulating hormone concentration</dd><dd>HP:0001410, Decreased liver function - HP:0002925, Elevated circulating thyroid-stimulating hormone concentration</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0002925, Elevated circulating thyroid-stimulating hormone concentration</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0001642, Pulmonic stenosis</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000842, Hyperinsulinemia</dd><dd>HP:0003073, Hypoalbuminemia - HP:0000842, Hyperinsulinemia</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0000842, Hyperinsulinemia</dd><dd>HP:0001873, Thrombocytopenia - HP:0001873, Thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - HP:0000842, Hyperinsulinemia</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0000842, Hyperinsulinemia</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001903, Anemia</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAGEC1%0DSMC5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMC5</a> and phenotypic similarity 0.601 to mouse mutant of SMC5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0005011, increased eosinophil cell number</dd><dd>HP:0001903, Anemia - MP:0002875, decreased erythrocyte cell number</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - MP:0005011, increased eosinophil cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0005016, decreased lymphocyte cell number</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0000186, decreased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia - MP:0003179, thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0005560, decreased circulating glucose level</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0000219, increased neutrophil cell number</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=8.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.090</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-141905936-A-C [0/1:0/0:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs80314937">rs80314937</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.090</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285879.5">MAGEC1:ENST00000285879.5:c.532A&gt;C:p.(Ile178Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.10462268</dd>
                                    
                                    <dd>REVEL: 0.015</dd>
                                    <dd>MVP: 0.105</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AFR: 0.3339%</dd>
                                    <dd>ExAC AMR: 0.0562%</dd>
                                    <dd>ExAC EAS: 0.0405%</dd>
                                    <dd>ExAC FIN: 0.0387%</dd>
                                    <dd>ExAC NFE: 0.0702%</dd>
                                    <dd>ExAC OTH: 0.4651%</dd>
                                    <dd>ExAC SAS: 0.0859%</dd>
                                    <dd>gnomAD_E_AFR: 0.2972%</dd>
                                    <dd>gnomAD_E_AMR: 0.0556%</dd>
                                    <dd>gnomAD_E_EAS: 0.0889%</dd>
                                    <dd>gnomAD_E_FIN: 0.0794%</dd>
                                    <dd>gnomAD_E_NFE: 0.1027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0943%</dd>
                                    <dd>gnomAD_E_SAS: 0.1411%</dd>
                                    <dd>gnomAD_G_AFR: 0.7056%</dd>
                                    <dd>gnomAD_G_AMR: 0.5168%</dd>
                                    <dd>gnomAD_G_FIN: 0.1143%</dd>
                                    <dd>gnomAD_G_NFE: 0.0921%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SNX29" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000048471">SNX29</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=8.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.175</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.442</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.175 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SNX29">SNX29</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0005333, decreased heart rate</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0004738, abnormal auditory brainstem response</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=8.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.175</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.442</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>16-12568499-C-G [0/1:0/1:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs757805840">rs757805840</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000566228.6">SNX29:ENST00000566228.6:c.2319-7C&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-12052013-C-T [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148525738">rs148525738</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.085</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000566228.6">SNX29:ENST00000566228.6:c.915C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1093%</dd>
                                    <dd>TOPMed: 0.0185%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC EAS: 0.4507%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.4812%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_EAS: 0.7398%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LONRF3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175556">LONRF3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=8.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.071</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.302 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LONRF3">LONRF3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0001554, increased circulating free fatty acids level</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0001554, increased circulating free fatty acids level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0001554, increased circulating free fatty acids level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LONRF3%0DBBS4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BBS4</a> and phenotypic similarity 0.313 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=110">Bardet-Biedl syndrome</a> associated with BBS4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001395, Hepatic fibrosis</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0001395, Hepatic fibrosis</dd><dd>HP:0001410, Decreased liver function - HP:0001395, Hepatic fibrosis</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001395, Hepatic fibrosis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0000822, Hypertension</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001395, Hepatic fibrosis</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001513, Obesity</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LONRF3%0DBBS4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BBS4</a> and phenotypic similarity 0.633 to mouse mutant of BBS4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0000599, enlarged liver</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0000599, enlarged liver</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0000609, abnormal liver physiology</dd><dd>HP:0001410, Decreased liver function - MP:0000609, abnormal liver physiology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0000609, abnormal liver physiology</dd><dd>HP:0001399, Hepatic failure - MP:0000609, abnormal liver physiology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis - MP:0005343, increased circulating aspartate transaminase level</dd><dd>HP:0003073, Hypoalbuminemia - MP:0002941, increased circulating alanine transaminase level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0001556, increased circulating HDL cholesterol level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0005344, increased circulating bilirubin level</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0000609, abnormal liver physiology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0000609, abnormal liver physiology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - MP:0003729, abnormal photoreceptor outer segment morphology</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0002628, hepatic steatosis</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - MP:0001262, decreased body weight</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LONRF3%0DBBS4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BBS4</a> and phenotypic similarity 0.269 to fish mutant of BBS4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0001410, Decreased liver function - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0002910, Elevated hepatic transaminase - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0001399, Hepatic failure - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - ZP:0006380, photoreceptor cell outer segment organization disrupted, abnormal</dd><dd>HP:0006568, Increased hepatic glycogen content - ZP:0018684, exocrine pancreas decreased area, abnormal</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=8.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.071</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>X-118978451-G-A [0/1:0/0:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140224937">rs140224937</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.071</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371628.8">LONRF3:ENST00000371628.8:c.924G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1873%</dd>
                                    <dd>TOPMed: 0.1322%</dd>
                                    <dd>UK10K: 0.0303%</dd>
                                    <dd>ESP AA: 0.2868%</dd>
                                    <dd>ESP All: 0.1041%</dd>
                                    <dd>ExAC AFR: 0.2820%</dd>
                                    <dd>ExAC AMR: 0.0322%</dd>
                                    <dd>ExAC EAS: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0125%</dd>
                                    <dd>ExAC SAS: 1.1387%</dd>
                                    <dd>gnomAD_E_AFR: 0.3189%</dd>
                                    <dd>gnomAD_E_AMR: 0.0114%</dd>
                                    <dd>gnomAD_E_EAS: 0.0078%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_OTH: 0.0496%</dd>
                                    <dd>gnomAD_E_SAS: 1.1526%</dd>
                                    <dd>gnomAD_G_AFR: 0.3787%</dd>
                                    <dd>gnomAD_G_OTH: 0.2789%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="VCX3A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169059">VCX3A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.040</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VCX3A%0DPARN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PARN</a> and phenotypic similarity 0.640 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1775">Dyskeratosis congenita</a> associated with PARN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001744, Splenomegaly</dd><dd>HP:0001903, Anemia - HP:0001903, Anemia</dd><dd>HP:0001541, Ascites - HP:0002240, Hepatomegaly</dd><dd>HP:0010972, Anemia of inadequate production - HP:0001903, Anemia</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - HP:0005528, Bone marrow hypocellularity</dd><dd>HP:0001396, Cholestasis - HP:0001399, Hepatic failure</dd><dd>HP:0001410, Decreased liver function - HP:0001399, Hepatic failure</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001399, Hepatic failure</dd><dd>HP:0000938, Osteopenia - HP:0000939, Osteoporosis</dd><dd>HP:0001653, Mitral regurgitation - HP:0100585, Telangiectasia of the skin</dd><dd>HP:0200114, Metabolic alkalosis - HP:0000819, Diabetes mellitus</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - HP:0001873, Thrombocytopenia</dd><dd>HP:0002151, Increased serum lactate - HP:0000819, Diabetes mellitus</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001394, Cirrhosis</dd><dd>HP:0004333, Bone-marrow foam cells - HP:0001874, Abnormality of neutrophils</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=VCX3A%0DPARN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PARN</a> and phenotypic similarity 0.324 to fish mutant of PARN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - ZP:0000451, hemopoiesis disrupted, abnormal</dd><dd>HP:0001903, Anemia - ZP:0000451, hemopoiesis disrupted, abnormal</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production - ZP:0000451, hemopoiesis disrupted, abnormal</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - ZP:0000451, hemopoiesis disrupted, abnormal</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia - ZP:0000451, hemopoiesis disrupted, abnormal</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells - ZP:0000451, hemopoiesis disrupted, abnormal</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.040</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-6533948-C-T [0/1:0/0:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs753646427">rs753646427</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.040</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381089.7">VCX3A:ENST00000381089.7:c.358G&gt;A:p.(Val120Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.043077525</dd>
                                    
                                    <dd>REVEL: 0.012</dd>
                                    <dd>MVP: 0.043</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AFR: 0.0189%</dd>
                                    <dd>gnomAD_E_EAS: 0.0125%</dd>
                                    <dd>gnomAD_E_FIN: 0.0419%</dd>
                                    <dd>gnomAD_E_NFE: 0.0192%</dd>
                                    <dd>gnomAD_E_OTH: 0.0375%</dd>
                                    <dd>gnomAD_G_AFR: 0.1553%</dd>
                                    <dd>gnomAD_G_AMR: 0.3960%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NBPF10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000271425">NBPF10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.600</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.600</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-146121575-G-C [0/1:0/0:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1299657693">rs1299657693</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.600</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583866.9">NBPF10:ENST00000583866.9:c.2681C&gt;G:p.(Thr894Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.600</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-146069569-C-T [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs374175846">rs374175846</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2_Supporting]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.600</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583866.9">NBPF10:ENST00000583866.9:c.10784G&gt;A:p.(Arg3595His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MUC17" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169876">MUC17</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.014</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC17%0DGALNT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GALNT1</a> and phenotypic similarity 0.630 to mouse mutant of GALNT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0001903, Anemia - MP:0002607, decreased basophil cell number</dd><dd>HP:0001541, Ascites - MP:0005606, increased bleeding time</dd><dd>HP:0010972, Anemia of inadequate production - MP:0005012, decreased eosinophil cell number</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology - MP:0002607, decreased basophil cell number</dd><dd>HP:0001396, Cholestasis - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0001410, Decreased liver function - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0001399, Hepatic failure - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0006047, aortic valve regurgitation</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia - MP:0005419, decreased circulating serum albumin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0005419, decreased circulating serum albumin level</dd><dd>HP:0001873, Thrombocytopenia - MP:0002607, decreased basophil cell number</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0008482, decreased spleen germinal center number</dd><dd>HP:0004333, Bone-marrow foam cells - MP:0010470, dilated ascending aorta</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.00 (p-value: 1.000000)</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.014</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-101037135-T-A [0/1:./.:0/0]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs371070576">rs371070576</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <b>[PM2_Supporting, BP4_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.014</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306151.9">MUC17:ENST00000306151.9:c.5719T&gt;A:p.(Ser1907Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.013882241</dd>
                                    
                                    <dd>REVEL: 0.006</dd>
                                    <dd>MVP: 0.014</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCN8A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196876">SCN8A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.327 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">Non-specific early-onset epileptic encephalopathy</a> associated with SCN8A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - HP:0000648, Optic atrophy</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001508, Failure to thrive</dd></dl><dl><dt>Phenotypic similarity 0.355 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SCN8A">SCN8A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0004952, increased spleen weight</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - MP:0004952, increased spleen weight</dd><dd>HP:0010972, Anemia of inadequate production - MP:0004952, increased spleen weight</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0004952, increased spleen weight</dd><dd>HP:0001410, Decreased liver function - MP:0004952, increased spleen weight</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0004952, increased spleen weight</dd><dd>HP:0001399, Hepatic failure - MP:0004952, increased spleen weight</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0004952, increased spleen weight</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0004952, increased spleen weight</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - MP:0004952, increased spleen weight</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Phenotypic similarity 0.202 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SCN8A">SCN8A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly -</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis -</dd><dd>HP:0001410, Decreased liver function -</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure -</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation -</dd><dd>HP:0200114, Metabolic alkalosis -</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration -</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate -</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology - ZP:0000834, neuron differentiation process quality, abnormal</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN8A%0DSCNN1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SCNN1B</a> and phenotypic similarity 0.601 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=171876">Generalized pseudohypoaldosteronism type 1</a> associated with SCNN1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - HP:0001081, Cholelithiasis</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites - HP:0001944, Dehydration</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - HP:0001081, Cholelithiasis</dd><dd>HP:0001410, Decreased liver function - HP:0001081, Cholelithiasis</dd><dd>HP:0002910, Elevated hepatic transaminase -</dd><dd>HP:0001399, Hepatic failure - HP:0001081, Cholelithiasis</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - HP:0031274, Hypovolemic shock</dd><dd>HP:0200114, Metabolic alkalosis - HP:0001942, Metabolic acidosis</dd><dd>HP:0003073, Hypoalbuminemia - HP:0000848, Increased circulating renin level</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - HP:0040085, Abnormal circulating aldosterone</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - HP:0001942, Metabolic acidosis</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration -</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration -</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content - HP:0001081, Cholelithiasis</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy - HP:0001531, Failure to thrive in infancy</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN8A%0DSCNN1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SCNN1B</a> and phenotypic similarity 0.439 to mouse mutant of SCNN1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0000007, Autosomal recessive inheritance -</dd><dd>HP:0001433, Hepatosplenomegaly - MP:0001625, cardiac hypertrophy</dd><dd>HP:0001903, Anemia -</dd><dd>HP:0001541, Ascites -</dd><dd>HP:0010972, Anemia of inadequate production -</dd><dd>HP:0005561, Abnormality of bone marrow cell morphology -</dd><dd>HP:0001396, Cholestasis - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0001410, Decreased liver function - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0002910, Elevated hepatic transaminase - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0001399, Hepatic failure - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0000938, Osteopenia -</dd><dd>HP:0001653, Mitral regurgitation - MP:0010127, hypervolemia</dd><dd>HP:0200114, Metabolic alkalosis - MP:0003028, alkalosis</dd><dd>HP:0003073, Hypoalbuminemia -</dd><dd>HP:0003233, Decreased HDL cholesterol concentration - MP:0002667, decreased circulating aldosterone level</dd><dd>HP:0001873, Thrombocytopenia -</dd><dd>HP:0002151, Increased serum lactate - MP:0003028, alkalosis</dd><dd>HP:0031964, Elevated circulating alanine aminotransferase concentration - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0031956, Elevated circulating aspartate aminotransferase concentration - MP:0001663, abnormal digestive system physiology</dd><dd>HP:0002366, Abnormal lower motor neuron morphology -</dd><dd>HP:0006568, Increased hepatic glycogen content -</dd><dd>HP:0004333, Bone-marrow foam cells -</dd><dd>HP:0001531, Failure to thrive in infancy -</dd></dl><dl><dt>PhenIX semantic similarity score: 0.41 (p-value: 0.856380)</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/614306">OMIM:614306</a> Cognitive impairment with or without cerebellar ataxia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/614558">OMIM:614558</a> Developmental and epileptic encephalopathy 13 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/617080">OMIM:617080</a> Seizures, benign familial infantile, 5 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618364">OMIM:618364</a> ?Myoclonus, familial, 2 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=306">ORPHA:306</a> Benign familial infantile epilepsy - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=31709">ORPHA:31709</a> Infantile convulsions and choreoathetosis - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Non-specific early-onset epileptic encephalopathy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.253</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.097</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-51663087-G-T [1/1:0/1:0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs371712630">rs371712630</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=192454%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.097</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354534.11">SCN8A:ENST00000354534.11:c.270G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000627620.5">SCN8A:ENST00000627620.5:c.270G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0625%</dd>
                                    <dd>TOPMed: 0.0625%</dd>
                                    <dd>ExAC EAS: 0.1739%</dd>
                                    <dd>ExAC SAS: 0.0062%</dd>
                                    <dd>gnomAD_E_EAS: 0.1218%</dd>
                                    <dd>gnomAD_E_OTH: 0.0549%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="https://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="https://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="https://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="https://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="https://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
